// lucide-angular/esm2020/aliases/suffixed.mjs
// AArrowDown aliases
export { default as AArrowDownIcon } from '../icons/a-arrow-down';
// AArrowUp aliases
export { default as AArrowUpIcon } from '../icons/a-arrow-up';
// ALargeSmall aliases
export { default as ALargeSmallIcon } from '../icons/a-large-small';
// Accessibility aliases
export { default as AccessibilityIcon } from '../icons/accessibility';
// Activity aliases
export { default as ActivityIcon } from '../icons/activity';
// AirVent aliases
export { default as AirVentIcon } from '../icons/air-vent';
// Airplay aliases
export { default as AirplayIcon } from '../icons/airplay';
// AlarmClockOff aliases
export { default as AlarmClockOffIcon } from '../icons/alarm-clock-off';
// AlarmClock aliases
export { default as AlarmClockIcon } from '../icons/alarm-clock';
// AlarmSmoke aliases
export { default as AlarmSmokeIcon } from '../icons/alarm-smoke';
// Album aliases
export { default as AlbumIcon } from '../icons/album';
// AlignCenterHorizontal aliases
export { default as AlignCenterHorizontalIcon } from '../icons/align-center-horizontal';
// AlignCenterVertical aliases
export { default as AlignCenterVerticalIcon } from '../icons/align-center-vertical';
// AlignEndHorizontal aliases
export { default as AlignEndHorizontalIcon } from '../icons/align-end-horizontal';
// AlignEndVertical aliases
export { default as AlignEndVerticalIcon } from '../icons/align-end-vertical';
// AlignHorizontalDistributeCenter aliases
export { default as AlignHorizontalDistributeCenterIcon } from '../icons/align-horizontal-distribute-center';
// AlignHorizontalDistributeEnd aliases
export { default as AlignHorizontalDistributeEndIcon } from '../icons/align-horizontal-distribute-end';
// AlignHorizontalDistributeStart aliases
export { default as AlignHorizontalDistributeStartIcon } from '../icons/align-horizontal-distribute-start';
// AlignHorizontalJustifyCenter aliases
export { default as AlignHorizontalJustifyCenterIcon } from '../icons/align-horizontal-justify-center';
// AlignHorizontalJustifyEnd aliases
export { default as AlignHorizontalJustifyEndIcon } from '../icons/align-horizontal-justify-end';
// AlignHorizontalSpaceAround aliases
export { default as AlignHorizontalSpaceAroundIcon } from '../icons/align-horizontal-space-around';
// AlignHorizontalJustifyStart aliases
export { default as AlignHorizontalJustifyStartIcon } from '../icons/align-horizontal-justify-start';
// AlignHorizontalSpaceBetween aliases
export { default as AlignHorizontalSpaceBetweenIcon } from '../icons/align-horizontal-space-between';
// AlignStartHorizontal aliases
export { default as AlignStartHorizontalIcon } from '../icons/align-start-horizontal';
// AlignStartVertical aliases
export { default as AlignStartVerticalIcon } from '../icons/align-start-vertical';
// AlignVerticalDistributeCenter aliases
export { default as AlignVerticalDistributeCenterIcon } from '../icons/align-vertical-distribute-center';
// AlignVerticalDistributeEnd aliases
export { default as AlignVerticalDistributeEndIcon } from '../icons/align-vertical-distribute-end';
// AlignVerticalDistributeStart aliases
export { default as AlignVerticalDistributeStartIcon } from '../icons/align-vertical-distribute-start';
// AlignVerticalJustifyCenter aliases
export { default as AlignVerticalJustifyCenterIcon } from '../icons/align-vertical-justify-center';
// AlignVerticalJustifyEnd aliases
export { default as AlignVerticalJustifyEndIcon } from '../icons/align-vertical-justify-end';
// AlignVerticalJustifyStart aliases
export { default as AlignVerticalJustifyStartIcon } from '../icons/align-vertical-justify-start';
// AlignVerticalSpaceBetween aliases
export { default as AlignVerticalSpaceBetweenIcon } from '../icons/align-vertical-space-between';
// AlignVerticalSpaceAround aliases
export { default as AlignVerticalSpaceAroundIcon } from '../icons/align-vertical-space-around';
// Ambulance aliases
export { default as AmbulanceIcon } from '../icons/ambulance';
// Ampersand aliases
export { default as AmpersandIcon } from '../icons/ampersand';
// Ampersands aliases
export { default as AmpersandsIcon } from '../icons/ampersands';
// Amphora aliases
export { default as AmphoraIcon } from '../icons/amphora';
// Angry aliases
export { default as AngryIcon } from '../icons/angry';
// Anchor aliases
export { default as AnchorIcon } from '../icons/anchor';
// Annoyed aliases
export { default as AnnoyedIcon } from '../icons/annoyed';
// Antenna aliases
export { default as AntennaIcon } from '../icons/antenna';
// Anvil aliases
export { default as AnvilIcon } from '../icons/anvil';
// Aperture aliases
export { default as ApertureIcon } from '../icons/aperture';
// AppWindowMac aliases
export { default as AppWindowMacIcon } from '../icons/app-window-mac';
// Apple aliases
export { default as AppleIcon } from '../icons/apple';
// AppWindow aliases
export { default as AppWindowIcon } from '../icons/app-window';
// ArchiveX aliases
export { default as ArchiveXIcon } from '../icons/archive-x';
// ArchiveRestore aliases
export { default as ArchiveRestoreIcon } from '../icons/archive-restore';
// Archive aliases
export { default as ArchiveIcon } from '../icons/archive';
// Armchair aliases
export { default as ArmchairIcon } from '../icons/armchair';
// ArrowBigDownDash aliases
export { default as ArrowBigDownDashIcon } from '../icons/arrow-big-down-dash';
// ArrowBigDown aliases
export { default as ArrowBigDownIcon } from '../icons/arrow-big-down';
// ArrowBigLeftDash aliases
export { default as ArrowBigLeftDashIcon } from '../icons/arrow-big-left-dash';
// ArrowBigLeft aliases
export { default as ArrowBigLeftIcon } from '../icons/arrow-big-left';
// ArrowBigRightDash aliases
export { default as ArrowBigRightDashIcon } from '../icons/arrow-big-right-dash';
// ArrowBigRight aliases
export { default as ArrowBigRightIcon } from '../icons/arrow-big-right';
// ArrowBigUpDash aliases
export { default as ArrowBigUpDashIcon } from '../icons/arrow-big-up-dash';
// ArrowBigUp aliases
export { default as ArrowBigUpIcon } from '../icons/arrow-big-up';
// ArrowDownLeft aliases
export { default as ArrowDownLeftIcon } from '../icons/arrow-down-left';
// ArrowDownFromLine aliases
export { default as ArrowDownFromLineIcon } from '../icons/arrow-down-from-line';
// ArrowDownNarrowWide aliases
export { default as ArrowDownNarrowWideIcon } from '../icons/arrow-down-narrow-wide';
// ArrowDownRight aliases
export { default as ArrowDownRightIcon } from '../icons/arrow-down-right';
// ArrowDownToDot aliases
export { default as ArrowDownToDotIcon } from '../icons/arrow-down-to-dot';
// ArrowDownToLine aliases
export { default as ArrowDownToLineIcon } from '../icons/arrow-down-to-line';
// ArrowDownUp aliases
export { default as ArrowDownUpIcon } from '../icons/arrow-down-up';
// ArrowDown aliases
export { default as ArrowDownIcon } from '../icons/arrow-down';
// ArrowLeftFromLine aliases
export { default as ArrowLeftFromLineIcon } from '../icons/arrow-left-from-line';
// ArrowLeftRight aliases
export { default as ArrowLeftRightIcon } from '../icons/arrow-left-right';
// ArrowLeftToLine aliases
export { default as ArrowLeftToLineIcon } from '../icons/arrow-left-to-line';
// ArrowRightFromLine aliases
export { default as ArrowRightFromLineIcon } from '../icons/arrow-right-from-line';
// ArrowLeft aliases
export { default as ArrowLeftIcon } from '../icons/arrow-left';
// ArrowRightLeft aliases
export { default as ArrowRightLeftIcon } from '../icons/arrow-right-left';
// ArrowRightToLine aliases
export { default as ArrowRightToLineIcon } from '../icons/arrow-right-to-line';
// ArrowRight aliases
export { default as ArrowRightIcon } from '../icons/arrow-right';
// ArrowUpDown aliases
export { default as ArrowUpDownIcon } from '../icons/arrow-up-down';
// ArrowUpFromDot aliases
export { default as ArrowUpFromDotIcon } from '../icons/arrow-up-from-dot';
// ArrowUpFromLine aliases
export { default as ArrowUpFromLineIcon } from '../icons/arrow-up-from-line';
// ArrowUpLeft aliases
export { default as ArrowUpLeftIcon } from '../icons/arrow-up-left';
// ArrowUpRight aliases
export { default as ArrowUpRightIcon } from '../icons/arrow-up-right';
// ArrowUpToLine aliases
export { default as ArrowUpToLineIcon } from '../icons/arrow-up-to-line';
// ArrowUpWideNarrow aliases
export { default as ArrowUpWideNarrowIcon } from '../icons/arrow-up-wide-narrow';
// ArrowUp aliases
export { default as ArrowUpIcon } from '../icons/arrow-up';
// Atom aliases
export { default as AtomIcon } from '../icons/atom';
// ArrowsUpFromLine aliases
export { default as ArrowsUpFromLineIcon } from '../icons/arrows-up-from-line';
// Asterisk aliases
export { default as AsteriskIcon } from '../icons/asterisk';
// AtSign aliases
export { default as AtSignIcon } from '../icons/at-sign';
// AudioLines aliases
export { default as AudioLinesIcon } from '../icons/audio-lines';
// AudioWaveform aliases
export { default as AudioWaveformIcon } from '../icons/audio-waveform';
// Award aliases
export { default as AwardIcon } from '../icons/award';
// Axe aliases
export { default as AxeIcon } from '../icons/axe';
// Backpack aliases
export { default as BackpackIcon } from '../icons/backpack';
// Baby aliases
export { default as BabyIcon } from '../icons/baby';
// BadgeCent aliases
export { default as BadgeCentIcon } from '../icons/badge-cent';
// BadgeDollarSign aliases
export { default as BadgeDollarSignIcon } from '../icons/badge-dollar-sign';
// BadgeAlert aliases
export { default as BadgeAlertIcon } from '../icons/badge-alert';
// BadgeInfo aliases
export { default as BadgeInfoIcon } from '../icons/badge-info';
// BadgeEuro aliases
export { default as BadgeEuroIcon } from '../icons/badge-euro';
// BadgeIndianRupee aliases
export { default as BadgeIndianRupeeIcon } from '../icons/badge-indian-rupee';
// BadgeJapaneseYen aliases
export { default as BadgeJapaneseYenIcon } from '../icons/badge-japanese-yen';
// BadgePercent aliases
export { default as BadgePercentIcon } from '../icons/badge-percent';
// BadgeMinus aliases
export { default as BadgeMinusIcon } from '../icons/badge-minus';
// BadgeRussianRuble aliases
export { default as BadgeRussianRubleIcon } from '../icons/badge-russian-ruble';
// BadgePlus aliases
export { default as BadgePlusIcon } from '../icons/badge-plus';
// BadgePoundSterling aliases
export { default as BadgePoundSterlingIcon } from '../icons/badge-pound-sterling';
// BadgeSwissFranc aliases
export { default as BadgeSwissFrancIcon } from '../icons/badge-swiss-franc';
// BadgeTurkishLira aliases
export { default as BadgeTurkishLiraIcon } from '../icons/badge-turkish-lira';
// Badge aliases
export { default as BadgeIcon } from '../icons/badge';
// BadgeX aliases
export { default as BadgeXIcon } from '../icons/badge-x';
// BaggageClaim aliases
export { default as BaggageClaimIcon } from '../icons/baggage-claim';
// Ban aliases
export { default as BanIcon } from '../icons/ban';
// Banana aliases
export { default as BananaIcon } from '../icons/banana';
// Bandage aliases
export { default as BandageIcon } from '../icons/bandage';
// BanknoteArrowDown aliases
export { default as BanknoteArrowDownIcon } from '../icons/banknote-arrow-down';
// BanknoteArrowUp aliases
export { default as BanknoteArrowUpIcon } from '../icons/banknote-arrow-up';
// BanknoteX aliases
export { default as BanknoteXIcon } from '../icons/banknote-x';
// Banknote aliases
export { default as BanknoteIcon } from '../icons/banknote';
// Barcode aliases
export { default as BarcodeIcon } from '../icons/barcode';
// Barrel aliases
export { default as BarrelIcon } from '../icons/barrel';
// Baseline aliases
export { default as BaselineIcon } from '../icons/baseline';
// Bath aliases
export { default as BathIcon } from '../icons/bath';
// BatteryFull aliases
export { default as BatteryFullIcon } from '../icons/battery-full';
// BatteryCharging aliases
export { default as BatteryChargingIcon } from '../icons/battery-charging';
// BatteryLow aliases
export { default as BatteryLowIcon } from '../icons/battery-low';
// BatteryMedium aliases
export { default as BatteryMediumIcon } from '../icons/battery-medium';
// BatteryPlus aliases
export { default as BatteryPlusIcon } from '../icons/battery-plus';
// BatteryWarning aliases
export { default as BatteryWarningIcon } from '../icons/battery-warning';
// Beaker aliases
export { default as BeakerIcon } from '../icons/beaker';
// BeanOff aliases
export { default as BeanOffIcon } from '../icons/bean-off';
// Battery aliases
export { default as BatteryIcon } from '../icons/battery';
// Bean aliases
export { default as BeanIcon } from '../icons/bean';
// BedDouble aliases
export { default as BedDoubleIcon } from '../icons/bed-double';
// BedSingle aliases
export { default as BedSingleIcon } from '../icons/bed-single';
// Bed aliases
export { default as BedIcon } from '../icons/bed';
// Beef aliases
export { default as BeefIcon } from '../icons/beef';
// BeerOff aliases
export { default as BeerOffIcon } from '../icons/beer-off';
// Beer aliases
export { default as BeerIcon } from '../icons/beer';
// BellDot aliases
export { default as BellDotIcon } from '../icons/bell-dot';
// BellElectric aliases
export { default as BellElectricIcon } from '../icons/bell-electric';
// BellMinus aliases
export { default as BellMinusIcon } from '../icons/bell-minus';
// BellOff aliases
export { default as BellOffIcon } from '../icons/bell-off';
// BellRing aliases
export { default as BellRingIcon } from '../icons/bell-ring';
// BellPlus aliases
export { default as BellPlusIcon } from '../icons/bell-plus';
// BetweenVerticalEnd aliases
export { default as BetweenVerticalEndIcon } from '../icons/between-vertical-end';
// Bell aliases
export { default as BellIcon } from '../icons/bell';
// BicepsFlexed aliases
export { default as BicepsFlexedIcon } from '../icons/biceps-flexed';
// BetweenVerticalStart aliases
export { default as BetweenVerticalStartIcon } from '../icons/between-vertical-start';
// Binary aliases
export { default as BinaryIcon } from '../icons/binary';
// Binoculars aliases
export { default as BinocularsIcon } from '../icons/binoculars';
// Bike aliases
export { default as BikeIcon } from '../icons/bike';
// Biohazard aliases
export { default as BiohazardIcon } from '../icons/biohazard';
// Bird aliases
export { default as BirdIcon } from '../icons/bird';
// Bitcoin aliases
export { default as BitcoinIcon } from '../icons/bitcoin';
// Blend aliases
export { default as BlendIcon } from '../icons/blend';
// Blinds aliases
export { default as BlindsIcon } from '../icons/blinds';
// Blocks aliases
export { default as BlocksIcon } from '../icons/blocks';
// BluetoothConnected aliases
export { default as BluetoothConnectedIcon } from '../icons/bluetooth-connected';
// BluetoothOff aliases
export { default as BluetoothOffIcon } from '../icons/bluetooth-off';
// BluetoothSearching aliases
export { default as BluetoothSearchingIcon } from '../icons/bluetooth-searching';
// Bold aliases
export { default as BoldIcon } from '../icons/bold';
// Bluetooth aliases
export { default as BluetoothIcon } from '../icons/bluetooth';
// Bolt aliases
export { default as BoltIcon } from '../icons/bolt';
// Bone aliases
export { default as BoneIcon } from '../icons/bone';
// BookA aliases
export { default as BookAIcon } from '../icons/book-a';
// Bomb aliases
export { default as BombIcon } from '../icons/bomb';
// BookAlert aliases
export { default as BookAlertIcon } from '../icons/book-alert';
// BookAudio aliases
export { default as BookAudioIcon } from '../icons/book-audio';
// BookCheck aliases
export { default as BookCheckIcon } from '../icons/book-check';
// BookCopy aliases
export { default as BookCopyIcon } from '../icons/book-copy';
// BookDown aliases
export { default as BookDownIcon } from '../icons/book-down';
// BookHeadphones aliases
export { default as BookHeadphonesIcon } from '../icons/book-headphones';
// BookHeart aliases
export { default as BookHeartIcon } from '../icons/book-heart';
// BookImage aliases
export { default as BookImageIcon } from '../icons/book-image';
// BookKey aliases
export { default as BookKeyIcon } from '../icons/book-key';
// BookLock aliases
export { default as BookLockIcon } from '../icons/book-lock';
// BookOpenCheck aliases
export { default as BookOpenCheckIcon } from '../icons/book-open-check';
// BookMarked aliases
export { default as BookMarkedIcon } from '../icons/book-marked';
// BookOpenText aliases
export { default as BookOpenTextIcon } from '../icons/book-open-text';
// BookMinus aliases
export { default as BookMinusIcon } from '../icons/book-minus';
// BookOpen aliases
export { default as BookOpenIcon } from '../icons/book-open';
// BookPlus aliases
export { default as BookPlusIcon } from '../icons/book-plus';
// BookText aliases
export { default as BookTextIcon } from '../icons/book-text';
// BookType aliases
export { default as BookTypeIcon } from '../icons/book-type';
// BookUp2 aliases
export { default as BookUp2Icon } from '../icons/book-up-2';
// BookUser aliases
export { default as BookUserIcon } from '../icons/book-user';
// BookUp aliases
export { default as BookUpIcon } from '../icons/book-up';
// Book aliases
export { default as BookIcon } from '../icons/book';
// BookX aliases
export { default as BookXIcon } from '../icons/book-x';
// BookmarkCheck aliases
export { default as BookmarkCheckIcon } from '../icons/bookmark-check';
// BookmarkMinus aliases
export { default as BookmarkMinusIcon } from '../icons/bookmark-minus';
// BookmarkPlus aliases
export { default as BookmarkPlusIcon } from '../icons/bookmark-plus';
// BookmarkX aliases
export { default as BookmarkXIcon } from '../icons/bookmark-x';
// BoomBox aliases
export { default as BoomBoxIcon } from '../icons/boom-box';
// Bookmark aliases
export { default as BookmarkIcon } from '../icons/bookmark';
// BotMessageSquare aliases
export { default as BotMessageSquareIcon } from '../icons/bot-message-square';
// BotOff aliases
export { default as BotOffIcon } from '../icons/bot-off';
// Bot aliases
export { default as BotIcon } from '../icons/bot';
// BottleWine aliases
export { default as BottleWineIcon } from '../icons/bottle-wine';
// BowArrow aliases
export { default as BowArrowIcon } from '../icons/bow-arrow';
// Box aliases
export { default as BoxIcon } from '../icons/box';
// Boxes aliases
export { default as BoxesIcon } from '../icons/boxes';
// Brackets aliases
export { default as BracketsIcon } from '../icons/brackets';
// BrainCircuit aliases
export { default as BrainCircuitIcon } from '../icons/brain-circuit';
// Brain aliases
export { default as BrainIcon } from '../icons/brain';
// BrainCog aliases
export { default as BrainCogIcon } from '../icons/brain-cog';
// BrickWallFire aliases
export { default as BrickWallFireIcon } from '../icons/brick-wall-fire';
// BrickWallShield aliases
export { default as BrickWallShieldIcon } from '../icons/brick-wall-shield';
// BrickWall aliases
export { default as BrickWallIcon } from '../icons/brick-wall';
// BriefcaseConveyorBelt aliases
export { default as BriefcaseConveyorBeltIcon } from '../icons/briefcase-conveyor-belt';
// BriefcaseMedical aliases
export { default as BriefcaseMedicalIcon } from '../icons/briefcase-medical';
// Briefcase aliases
export { default as BriefcaseIcon } from '../icons/briefcase';
// BriefcaseBusiness aliases
export { default as BriefcaseBusinessIcon } from '../icons/briefcase-business';
// BrushCleaning aliases
export { default as BrushCleaningIcon } from '../icons/brush-cleaning';
// BringToFront aliases
export { default as BringToFrontIcon } from '../icons/bring-to-front';
// Brush aliases
export { default as BrushIcon } from '../icons/brush';
// Bubbles aliases
export { default as BubblesIcon } from '../icons/bubbles';
// BugPlay aliases
export { default as BugPlayIcon } from '../icons/bug-play';
// BugOff aliases
export { default as BugOffIcon } from '../icons/bug-off';
// Bug aliases
export { default as BugIcon } from '../icons/bug';
// Building2 aliases
export { default as Building2Icon } from '../icons/building-2';
// Building aliases
export { default as BuildingIcon } from '../icons/building';
// CableCar aliases
export { default as CableCarIcon } from '../icons/cable-car';
// Bus aliases
export { default as BusIcon } from '../icons/bus';
// BusFront aliases
export { default as BusFrontIcon } from '../icons/bus-front';
// Cable aliases
export { default as CableIcon } from '../icons/cable';
// CakeSlice aliases
export { default as CakeSliceIcon } from '../icons/cake-slice';
// Calculator aliases
export { default as CalculatorIcon } from '../icons/calculator';
// Cake aliases
export { default as CakeIcon } from '../icons/cake';
// Calendar1 aliases
export { default as Calendar1Icon } from '../icons/calendar-1';
// CalendarArrowDown aliases
export { default as CalendarArrowDownIcon } from '../icons/calendar-arrow-down';
// CalendarArrowUp aliases
export { default as CalendarArrowUpIcon } from '../icons/calendar-arrow-up';
// CalendarCheck aliases
export { default as CalendarCheckIcon } from '../icons/calendar-check';
// CalendarClock aliases
export { default as CalendarClockIcon } from '../icons/calendar-clock';
// CalendarCheck2 aliases
export { default as CalendarCheck2Icon } from '../icons/calendar-check-2';
// CalendarCog aliases
export { default as CalendarCogIcon } from '../icons/calendar-cog';
// CalendarDays aliases
export { default as CalendarDaysIcon } from '../icons/calendar-days';
// CalendarFold aliases
export { default as CalendarFoldIcon } from '../icons/calendar-fold';
// CalendarHeart aliases
export { default as CalendarHeartIcon } from '../icons/calendar-heart';
// CalendarMinus2 aliases
export { default as CalendarMinus2Icon } from '../icons/calendar-minus-2';
// CalendarMinus aliases
export { default as CalendarMinusIcon } from '../icons/calendar-minus';
// CalendarOff aliases
export { default as CalendarOffIcon } from '../icons/calendar-off';
// CalendarPlus2 aliases
export { default as CalendarPlus2Icon } from '../icons/calendar-plus-2';
// CalendarPlus aliases
export { default as CalendarPlusIcon } from '../icons/calendar-plus';
// CalendarRange aliases
export { default as CalendarRangeIcon } from '../icons/calendar-range';
// CalendarSearch aliases
export { default as CalendarSearchIcon } from '../icons/calendar-search';
// CalendarSync aliases
export { default as CalendarSyncIcon } from '../icons/calendar-sync';
// CalendarX2 aliases
export { default as CalendarX2Icon } from '../icons/calendar-x-2';
// CalendarX aliases
export { default as CalendarXIcon } from '../icons/calendar-x';
// Calendar aliases
export { default as CalendarIcon } from '../icons/calendar';
// CameraOff aliases
export { default as CameraOffIcon } from '../icons/camera-off';
// Camera aliases
export { default as CameraIcon } from '../icons/camera';
// CandyCane aliases
export { default as CandyCaneIcon } from '../icons/candy-cane';
// CandyOff aliases
export { default as CandyOffIcon } from '../icons/candy-off';
// Candy aliases
export { default as CandyIcon } from '../icons/candy';
// CaptionsOff aliases
export { default as CaptionsOffIcon } from '../icons/captions-off';
// Cannabis aliases
export { default as CannabisIcon } from '../icons/cannabis';
// CarFront aliases
export { default as CarFrontIcon } from '../icons/car-front';
// Car aliases
export { default as CarIcon } from '../icons/car';
// CarTaxiFront aliases
export { default as CarTaxiFrontIcon } from '../icons/car-taxi-front';
// CardSim aliases
export { default as CardSimIcon } from '../icons/card-sim';
// Caravan aliases
export { default as CaravanIcon } from '../icons/caravan';
// Carrot aliases
export { default as CarrotIcon } from '../icons/carrot';
// CaseLower aliases
export { default as CaseLowerIcon } from '../icons/case-lower';
// CaseSensitive aliases
export { default as CaseSensitiveIcon } from '../icons/case-sensitive';
// CassetteTape aliases
export { default as CassetteTapeIcon } from '../icons/cassette-tape';
// CaseUpper aliases
export { default as CaseUpperIcon } from '../icons/case-upper';
// Cast aliases
export { default as CastIcon } from '../icons/cast';
// Castle aliases
export { default as CastleIcon } from '../icons/castle';
// Cctv aliases
export { default as CctvIcon } from '../icons/cctv';
// Cat aliases
export { default as CatIcon } from '../icons/cat';
// ChartBarDecreasing aliases
export { default as ChartBarDecreasingIcon } from '../icons/chart-bar-decreasing';
// ChartBarIncreasing aliases
export { default as ChartBarIncreasingIcon } from '../icons/chart-bar-increasing';
// ChartBarStacked aliases
export { default as ChartBarStackedIcon } from '../icons/chart-bar-stacked';
// ChartColumnDecreasing aliases
export { default as ChartColumnDecreasingIcon } from '../icons/chart-column-decreasing';
// ChartColumnStacked aliases
export { default as ChartColumnStackedIcon } from '../icons/chart-column-stacked';
// ChartGantt aliases
export { default as ChartGanttIcon } from '../icons/chart-gantt';
// ChartNetwork aliases
export { default as ChartNetworkIcon } from '../icons/chart-network';
// ChartNoAxesColumnDecreasing aliases
export { default as ChartNoAxesColumnDecreasingIcon } from '../icons/chart-no-axes-column-decreasing';
// ChartNoAxesCombined aliases
export { default as ChartNoAxesCombinedIcon } from '../icons/chart-no-axes-combined';
// ChartSpline aliases
export { default as ChartSplineIcon } from '../icons/chart-spline';
// CheckLine aliases
export { default as CheckLineIcon } from '../icons/check-line';
// CheckCheck aliases
export { default as CheckCheckIcon } from '../icons/check-check';
// Check aliases
export { default as CheckIcon } from '../icons/check';
// ChefHat aliases
export { default as ChefHatIcon } from '../icons/chef-hat';
// Cherry aliases
export { default as CherryIcon } from '../icons/cherry';
// ChevronDown aliases
export { default as ChevronDownIcon } from '../icons/chevron-down';
// ChevronFirst aliases
export { default as ChevronFirstIcon } from '../icons/chevron-first';
// ChevronLast aliases
export { default as ChevronLastIcon } from '../icons/chevron-last';
// ChevronLeft aliases
export { default as ChevronLeftIcon } from '../icons/chevron-left';
// ChevronRight aliases
export { default as ChevronRightIcon } from '../icons/chevron-right';
// ChevronUp aliases
export { default as ChevronUpIcon } from '../icons/chevron-up';
// ChevronsDownUp aliases
export { default as ChevronsDownUpIcon } from '../icons/chevrons-down-up';
// ChevronsDown aliases
export { default as ChevronsDownIcon } from '../icons/chevrons-down';
// ChevronsLeftRightEllipsis aliases
export { default as ChevronsLeftRightEllipsisIcon } from '../icons/chevrons-left-right-ellipsis';
// ChevronsLeftRight aliases
export { default as ChevronsLeftRightIcon } from '../icons/chevrons-left-right';
// ChevronsLeft aliases
export { default as ChevronsLeftIcon } from '../icons/chevrons-left';
// ChevronsRightLeft aliases
export { default as ChevronsRightLeftIcon } from '../icons/chevrons-right-left';
// ChevronsRight aliases
export { default as ChevronsRightIcon } from '../icons/chevrons-right';
// ChevronsUpDown aliases
export { default as ChevronsUpDownIcon } from '../icons/chevrons-up-down';
// ChevronsUp aliases
export { default as ChevronsUpIcon } from '../icons/chevrons-up';
// Church aliases
export { default as ChurchIcon } from '../icons/church';
// Cigarette aliases
export { default as CigaretteIcon } from '../icons/cigarette';
// CigaretteOff aliases
export { default as CigaretteOffIcon } from '../icons/cigarette-off';
// CircleDashed aliases
export { default as CircleDashedIcon } from '../icons/circle-dashed';
// CircleDollarSign aliases
export { default as CircleDollarSignIcon } from '../icons/circle-dollar-sign';
// CircleDotDashed aliases
export { default as CircleDotDashedIcon } from '../icons/circle-dot-dashed';
// CircleDot aliases
export { default as CircleDotIcon } from '../icons/circle-dot';
// CircleEllipsis aliases
export { default as CircleEllipsisIcon } from '../icons/circle-ellipsis';
// CircleEqual aliases
export { default as CircleEqualIcon } from '../icons/circle-equal';
// CircleFadingArrowUp aliases
export { default as CircleFadingArrowUpIcon } from '../icons/circle-fading-arrow-up';
// CircleFadingPlus aliases
export { default as CircleFadingPlusIcon } from '../icons/circle-fading-plus';
// CircleOff aliases
export { default as CircleOffIcon } from '../icons/circle-off';
// CircleSlash aliases
export { default as CircleSlashIcon } from '../icons/circle-slash';
// CircleSmall aliases
export { default as CircleSmallIcon } from '../icons/circle-small';
// CirclePoundSterling aliases
export { default as CirclePoundSterlingIcon } from '../icons/circle-pound-sterling';
// Circle aliases
export { default as CircleIcon } from '../icons/circle';
// CircleStar aliases
export { default as CircleStarIcon } from '../icons/circle-star';
// CircuitBoard aliases
export { default as CircuitBoardIcon } from '../icons/circuit-board';
// Citrus aliases
export { default as CitrusIcon } from '../icons/citrus';
// Clapperboard aliases
export { default as ClapperboardIcon } from '../icons/clapperboard';
// ClipboardCheck aliases
export { default as ClipboardCheckIcon } from '../icons/clipboard-check';
// ClipboardClock aliases
export { default as ClipboardClockIcon } from '../icons/clipboard-clock';
// ClipboardCopy aliases
export { default as ClipboardCopyIcon } from '../icons/clipboard-copy';
// ClipboardList aliases
export { default as ClipboardListIcon } from '../icons/clipboard-list';
// ClipboardMinus aliases
export { default as ClipboardMinusIcon } from '../icons/clipboard-minus';
// ClipboardPaste aliases
export { default as ClipboardPasteIcon } from '../icons/clipboard-paste';
// ClipboardPlus aliases
export { default as ClipboardPlusIcon } from '../icons/clipboard-plus';
// ClipboardX aliases
export { default as ClipboardXIcon } from '../icons/clipboard-x';
// ClipboardType aliases
export { default as ClipboardTypeIcon } from '../icons/clipboard-type';
// Clipboard aliases
export { default as ClipboardIcon } from '../icons/clipboard';
// Clock1 aliases
export { default as Clock1Icon } from '../icons/clock-1';
// Clock11 aliases
export { default as Clock11Icon } from '../icons/clock-11';
// Clock10 aliases
export { default as Clock10Icon } from '../icons/clock-10';
// Clock12 aliases
export { default as Clock12Icon } from '../icons/clock-12';
// Clock2 aliases
export { default as Clock2Icon } from '../icons/clock-2';
// Clock3 aliases
export { default as Clock3Icon } from '../icons/clock-3';
// Clock4 aliases
export { default as Clock4Icon } from '../icons/clock-4';
// Clock5 aliases
export { default as Clock5Icon } from '../icons/clock-5';
// Clock7 aliases
export { default as Clock7Icon } from '../icons/clock-7';
// Clock8 aliases
export { default as Clock8Icon } from '../icons/clock-8';
// Clock6 aliases
export { default as Clock6Icon } from '../icons/clock-6';
// Clock9 aliases
export { default as Clock9Icon } from '../icons/clock-9';
// ClockAlert aliases
export { default as ClockAlertIcon } from '../icons/clock-alert';
// ClockArrowDown aliases
export { default as ClockArrowDownIcon } from '../icons/clock-arrow-down';
// ClockFading aliases
export { default as ClockFadingIcon } from '../icons/clock-fading';
// ClockArrowUp aliases
export { default as ClockArrowUpIcon } from '../icons/clock-arrow-up';
// ClockPlus aliases
export { default as ClockPlusIcon } from '../icons/clock-plus';
// Clock aliases
export { default as ClockIcon } from '../icons/clock';
// CloudAlert aliases
export { default as CloudAlertIcon } from '../icons/cloud-alert';
// ClosedCaption aliases
export { default as ClosedCaptionIcon } from '../icons/closed-caption';
// CloudCog aliases
export { default as CloudCogIcon } from '../icons/cloud-cog';
// CloudDrizzle aliases
export { default as CloudDrizzleIcon } from '../icons/cloud-drizzle';
// CloudCheck aliases
export { default as CloudCheckIcon } from '../icons/cloud-check';
// CloudFog aliases
export { default as CloudFogIcon } from '../icons/cloud-fog';
// CloudLightning aliases
export { default as CloudLightningIcon } from '../icons/cloud-lightning';
// CloudHail aliases
export { default as CloudHailIcon } from '../icons/cloud-hail';
// CloudMoonRain aliases
export { default as CloudMoonRainIcon } from '../icons/cloud-moon-rain';
// CloudRain aliases
export { default as CloudRainIcon } from '../icons/cloud-rain';
// CloudOff aliases
export { default as CloudOffIcon } from '../icons/cloud-off';
// CloudMoon aliases
export { default as CloudMoonIcon } from '../icons/cloud-moon';
// CloudRainWind aliases
export { default as CloudRainWindIcon } from '../icons/cloud-rain-wind';
// CloudSnow aliases
export { default as CloudSnowIcon } from '../icons/cloud-snow';
// CloudSunRain aliases
export { default as CloudSunRainIcon } from '../icons/cloud-sun-rain';
// CloudSun aliases
export { default as CloudSunIcon } from '../icons/cloud-sun';
// Cloudy aliases
export { default as CloudyIcon } from '../icons/cloudy';
// Cloud aliases
export { default as CloudIcon } from '../icons/cloud';
// Club aliases
export { default as ClubIcon } from '../icons/club';
// Clover aliases
export { default as CloverIcon } from '../icons/clover';
// Code aliases
export { default as CodeIcon } from '../icons/code';
// Coffee aliases
export { default as CoffeeIcon } from '../icons/coffee';
// Codepen aliases
export { default as CodepenIcon } from '../icons/codepen';
// Codesandbox aliases
export { default as CodesandboxIcon } from '../icons/codesandbox';
// Cog aliases
export { default as CogIcon } from '../icons/cog';
// Coins aliases
export { default as CoinsIcon } from '../icons/coins';
// Columns4 aliases
export { default as Columns4Icon } from '../icons/columns-4';
// Combine aliases
export { default as CombineIcon } from '../icons/combine';
// Command aliases
export { default as CommandIcon } from '../icons/command';
// Compass aliases
export { default as CompassIcon } from '../icons/compass';
// Component aliases
export { default as ComponentIcon } from '../icons/component';
// Computer aliases
export { default as ComputerIcon } from '../icons/computer';
// ConciergeBell aliases
export { default as ConciergeBellIcon } from '../icons/concierge-bell';
// Construction aliases
export { default as ConstructionIcon } from '../icons/construction';
// Cone aliases
export { default as ConeIcon } from '../icons/cone';
// Contact aliases
export { default as ContactIcon } from '../icons/contact';
// Container aliases
export { default as ContainerIcon } from '../icons/container';
// Cookie aliases
export { default as CookieIcon } from '../icons/cookie';
// CookingPot aliases
export { default as CookingPotIcon } from '../icons/cooking-pot';
// Contrast aliases
export { default as ContrastIcon } from '../icons/contrast';
// CopyCheck aliases
export { default as CopyCheckIcon } from '../icons/copy-check';
// CopyMinus aliases
export { default as CopyMinusIcon } from '../icons/copy-minus';
// CopyPlus aliases
export { default as CopyPlusIcon } from '../icons/copy-plus';
// CopySlash aliases
export { default as CopySlashIcon } from '../icons/copy-slash';
// CopyX aliases
export { default as CopyXIcon } from '../icons/copy-x';
// Copy aliases
export { default as CopyIcon } from '../icons/copy';
// Copyleft aliases
export { default as CopyleftIcon } from '../icons/copyleft';
// CornerDownLeft aliases
export { default as CornerDownLeftIcon } from '../icons/corner-down-left';
// Copyright aliases
export { default as CopyrightIcon } from '../icons/copyright';
// CornerDownRight aliases
export { default as CornerDownRightIcon } from '../icons/corner-down-right';
// CornerLeftDown aliases
export { default as CornerLeftDownIcon } from '../icons/corner-left-down';
// CornerLeftUp aliases
export { default as CornerLeftUpIcon } from '../icons/corner-left-up';
// CornerRightDown aliases
export { default as CornerRightDownIcon } from '../icons/corner-right-down';
// CornerRightUp aliases
export { default as CornerRightUpIcon } from '../icons/corner-right-up';
// CornerUpLeft aliases
export { default as CornerUpLeftIcon } from '../icons/corner-up-left';
// CornerUpRight aliases
export { default as CornerUpRightIcon } from '../icons/corner-up-right';
// Cpu aliases
export { default as CpuIcon } from '../icons/cpu';
// CreditCard aliases
export { default as CreditCardIcon } from '../icons/credit-card';
// CreativeCommons aliases
export { default as CreativeCommonsIcon } from '../icons/creative-commons';
// Croissant aliases
export { default as CroissantIcon } from '../icons/croissant';
// Crop aliases
export { default as CropIcon } from '../icons/crop';
// Cross aliases
export { default as CrossIcon } from '../icons/cross';
// Crown aliases
export { default as CrownIcon } from '../icons/crown';
// Crosshair aliases
export { default as CrosshairIcon } from '../icons/crosshair';
// Cuboid aliases
export { default as CuboidIcon } from '../icons/cuboid';
// Currency aliases
export { default as CurrencyIcon } from '../icons/currency';
// CupSoda aliases
export { default as CupSodaIcon } from '../icons/cup-soda';
// Cylinder aliases
export { default as CylinderIcon } from '../icons/cylinder';
// Dam aliases
export { default as DamIcon } from '../icons/dam';
// DatabaseBackup aliases
export { default as DatabaseBackupIcon } from '../icons/database-backup';
// Database aliases
export { default as DatabaseIcon } from '../icons/database';
// DatabaseZap aliases
export { default as DatabaseZapIcon } from '../icons/database-zap';
// DecimalsArrowLeft aliases
export { default as DecimalsArrowLeftIcon } from '../icons/decimals-arrow-left';
// DecimalsArrowRight aliases
export { default as DecimalsArrowRightIcon } from '../icons/decimals-arrow-right';
// Delete aliases
export { default as DeleteIcon } from '../icons/delete';
// Diameter aliases
export { default as DiameterIcon } from '../icons/diameter';
// Dessert aliases
export { default as DessertIcon } from '../icons/dessert';
// DiamondMinus aliases
export { default as DiamondMinusIcon } from '../icons/diamond-minus';
// DiamondPlus aliases
export { default as DiamondPlusIcon } from '../icons/diamond-plus';
// Diamond aliases
export { default as DiamondIcon } from '../icons/diamond';
// Dice2 aliases
export { default as Dice2Icon } from '../icons/dice-2';
// Dice3 aliases
export { default as Dice3Icon } from '../icons/dice-3';
// Dice1 aliases
export { default as Dice1Icon } from '../icons/dice-1';
// Dice4 aliases
export { default as Dice4Icon } from '../icons/dice-4';
// Dice6 aliases
export { default as Dice6Icon } from '../icons/dice-6';
// Dice5 aliases
export { default as Dice5Icon } from '../icons/dice-5';
// Dices aliases
export { default as DicesIcon } from '../icons/dices';
// Disc2 aliases
export { default as Disc2Icon } from '../icons/disc-2';
// Diff aliases
export { default as DiffIcon } from '../icons/diff';
// Disc3 aliases
export { default as Disc3Icon } from '../icons/disc-3';
// DiscAlbum aliases
export { default as DiscAlbumIcon } from '../icons/disc-album';
// Divide aliases
export { default as DivideIcon } from '../icons/divide';
// Disc aliases
export { default as DiscIcon } from '../icons/disc';
// DnaOff aliases
export { default as DnaOffIcon } from '../icons/dna-off';
// Dna aliases
export { default as DnaIcon } from '../icons/dna';
// Dock aliases
export { default as DockIcon } from '../icons/dock';
// Dog aliases
export { default as DogIcon } from '../icons/dog';
// DollarSign aliases
export { default as DollarSignIcon } from '../icons/dollar-sign';
// Donut aliases
export { default as DonutIcon } from '../icons/donut';
// DoorClosedLocked aliases
export { default as DoorClosedLockedIcon } from '../icons/door-closed-locked';
// DoorClosed aliases
export { default as DoorClosedIcon } from '../icons/door-closed';
// Dot aliases
export { default as DotIcon } from '../icons/dot';
// Download aliases
export { default as DownloadIcon } from '../icons/download';
// DraftingCompass aliases
export { default as DraftingCompassIcon } from '../icons/drafting-compass';
// Drama aliases
export { default as DramaIcon } from '../icons/drama';
// DoorOpen aliases
export { default as DoorOpenIcon } from '../icons/door-open';
// Dribbble aliases
export { default as DribbbleIcon } from '../icons/dribbble';
// Drill aliases
export { default as DrillIcon } from '../icons/drill';
// DropletOff aliases
export { default as DropletOffIcon } from '../icons/droplet-off';
// Droplet aliases
export { default as DropletIcon } from '../icons/droplet';
// Drone aliases
export { default as DroneIcon } from '../icons/drone';
// Droplets aliases
export { default as DropletsIcon } from '../icons/droplets';
// Drum aliases
export { default as DrumIcon } from '../icons/drum';
// Drumstick aliases
export { default as DrumstickIcon } from '../icons/drumstick';
// Dumbbell aliases
export { default as DumbbellIcon } from '../icons/dumbbell';
// Ear aliases
export { default as EarIcon } from '../icons/ear';
// EarOff aliases
export { default as EarOffIcon } from '../icons/ear-off';
// EarthLock aliases
export { default as EarthLockIcon } from '../icons/earth-lock';
// Eclipse aliases
export { default as EclipseIcon } from '../icons/eclipse';
// EggFried aliases
export { default as EggFriedIcon } from '../icons/egg-fried';
// Egg aliases
export { default as EggIcon } from '../icons/egg';
// EggOff aliases
export { default as EggOffIcon } from '../icons/egg-off';
// EqualNot aliases
export { default as EqualNotIcon } from '../icons/equal-not';
// EqualApproximately aliases
export { default as EqualApproximatelyIcon } from '../icons/equal-approximately';
// Eraser aliases
export { default as EraserIcon } from '../icons/eraser';
// Equal aliases
export { default as EqualIcon } from '../icons/equal';
// EthernetPort aliases
export { default as EthernetPortIcon } from '../icons/ethernet-port';
// EvCharger aliases
export { default as EvChargerIcon } from '../icons/ev-charger';
// Euro aliases
export { default as EuroIcon } from '../icons/euro';
// Expand aliases
export { default as ExpandIcon } from '../icons/expand';
// EyeClosed aliases
export { default as EyeClosedIcon } from '../icons/eye-closed';
// EyeOff aliases
export { default as EyeOffIcon } from '../icons/eye-off';
// ExternalLink aliases
export { default as ExternalLinkIcon } from '../icons/external-link';
// Eye aliases
export { default as EyeIcon } from '../icons/eye';
// Facebook aliases
export { default as FacebookIcon } from '../icons/facebook';
// Factory aliases
export { default as FactoryIcon } from '../icons/factory';
// Fan aliases
export { default as FanIcon } from '../icons/fan';
// FastForward aliases
export { default as FastForwardIcon } from '../icons/fast-forward';
// Feather aliases
export { default as FeatherIcon } from '../icons/feather';
// Fence aliases
export { default as FenceIcon } from '../icons/fence';
// FerrisWheel aliases
export { default as FerrisWheelIcon } from '../icons/ferris-wheel';
// Figma aliases
export { default as FigmaIcon } from '../icons/figma';
// FileArchive aliases
export { default as FileArchiveIcon } from '../icons/file-archive';
// FileAudio2 aliases
export { default as FileAudio2Icon } from '../icons/file-audio-2';
// FileAudio aliases
export { default as FileAudioIcon } from '../icons/file-audio';
// FileBadge2 aliases
export { default as FileBadge2Icon } from '../icons/file-badge-2';
// FileBadge aliases
export { default as FileBadgeIcon } from '../icons/file-badge';
// FileBox aliases
export { default as FileBoxIcon } from '../icons/file-box';
// FileCheck aliases
export { default as FileCheckIcon } from '../icons/file-check';
// FileCheck2 aliases
export { default as FileCheck2Icon } from '../icons/file-check-2';
// FileClock aliases
export { default as FileClockIcon } from '../icons/file-clock';
// FileCode2 aliases
export { default as FileCode2Icon } from '../icons/file-code-2';
// FileCode aliases
export { default as FileCodeIcon } from '../icons/file-code';
// FileDiff aliases
export { default as FileDiffIcon } from '../icons/file-diff';
// FileDigit aliases
export { default as FileDigitIcon } from '../icons/file-digit';
// FileDown aliases
export { default as FileDownIcon } from '../icons/file-down';
// FileHeart aliases
export { default as FileHeartIcon } from '../icons/file-heart';
// FileJson2 aliases
export { default as FileJson2Icon } from '../icons/file-json-2';
// FileImage aliases
export { default as FileImageIcon } from '../icons/file-image';
// FileInput aliases
export { default as FileInputIcon } from '../icons/file-input';
// FileJson aliases
export { default as FileJsonIcon } from '../icons/file-json';
// FileKey aliases
export { default as FileKeyIcon } from '../icons/file-key';
// FileKey2 aliases
export { default as FileKey2Icon } from '../icons/file-key-2';
// FileLock2 aliases
export { default as FileLock2Icon } from '../icons/file-lock-2';
// FileLock aliases
export { default as FileLockIcon } from '../icons/file-lock';
// FileMinus aliases
export { default as FileMinusIcon } from '../icons/file-minus';
// FileMinus2 aliases
export { default as FileMinus2Icon } from '../icons/file-minus-2';
// FileMusic aliases
export { default as FileMusicIcon } from '../icons/file-music';
// FileOutput aliases
export { default as FileOutputIcon } from '../icons/file-output';
// FilePlus aliases
export { default as FilePlusIcon } from '../icons/file-plus';
// FileScan aliases
export { default as FileScanIcon } from '../icons/file-scan';
// FilePlus2 aliases
export { default as FilePlus2Icon } from '../icons/file-plus-2';
// FileSearch2 aliases
export { default as FileSearch2Icon } from '../icons/file-search-2';
// FileSearch aliases
export { default as FileSearchIcon } from '../icons/file-search';
// FileSliders aliases
export { default as FileSlidersIcon } from '../icons/file-sliders';
// FileSpreadsheet aliases
export { default as FileSpreadsheetIcon } from '../icons/file-spreadsheet';
// FileStack aliases
export { default as FileStackIcon } from '../icons/file-stack';
// FileSymlink aliases
export { default as FileSymlinkIcon } from '../icons/file-symlink';
// FileTerminal aliases
export { default as FileTerminalIcon } from '../icons/file-terminal';
// FileText aliases
export { default as FileTextIcon } from '../icons/file-text';
// FileType2 aliases
export { default as FileType2Icon } from '../icons/file-type-2';
// FileType aliases
export { default as FileTypeIcon } from '../icons/file-type';
// FileUser aliases
export { default as FileUserIcon } from '../icons/file-user';
// FileUp aliases
export { default as FileUpIcon } from '../icons/file-up';
// FileVolume2 aliases
export { default as FileVolume2Icon } from '../icons/file-volume-2';
// FileVolume aliases
export { default as FileVolumeIcon } from '../icons/file-volume';
// FileWarning aliases
export { default as FileWarningIcon } from '../icons/file-warning';
// FileX2 aliases
export { default as FileX2Icon } from '../icons/file-x-2';
// FileX aliases
export { default as FileXIcon } from '../icons/file-x';
// File aliases
export { default as FileIcon } from '../icons/file';
// Files aliases
export { default as FilesIcon } from '../icons/files';
// Film aliases
export { default as FilmIcon } from '../icons/film';
// Fingerprint aliases
export { default as FingerprintIcon } from '../icons/fingerprint';
// FishOff aliases
export { default as FishOffIcon } from '../icons/fish-off';
// FireExtinguisher aliases
export { default as FireExtinguisherIcon } from '../icons/fire-extinguisher';
// FishSymbol aliases
export { default as FishSymbolIcon } from '../icons/fish-symbol';
// FlagOff aliases
export { default as FlagOffIcon } from '../icons/flag-off';
// Fish aliases
export { default as FishIcon } from '../icons/fish';
// FlagTriangleLeft aliases
export { default as FlagTriangleLeftIcon } from '../icons/flag-triangle-left';
// FlagTriangleRight aliases
export { default as FlagTriangleRightIcon } from '../icons/flag-triangle-right';
// FlameKindling aliases
export { default as FlameKindlingIcon } from '../icons/flame-kindling';
// Flame aliases
export { default as FlameIcon } from '../icons/flame';
// Flashlight aliases
export { default as FlashlightIcon } from '../icons/flashlight';
// Flag aliases
export { default as FlagIcon } from '../icons/flag';
// FlashlightOff aliases
export { default as FlashlightOffIcon } from '../icons/flashlight-off';
// FlaskConicalOff aliases
export { default as FlaskConicalOffIcon } from '../icons/flask-conical-off';
// FlaskConical aliases
export { default as FlaskConicalIcon } from '../icons/flask-conical';
// FlaskRound aliases
export { default as FlaskRoundIcon } from '../icons/flask-round';
// FlipHorizontal2 aliases
export { default as FlipHorizontal2Icon } from '../icons/flip-horizontal-2';
// FlipHorizontal aliases
export { default as FlipHorizontalIcon } from '../icons/flip-horizontal';
// FlipVertical aliases
export { default as FlipVerticalIcon } from '../icons/flip-vertical';
// FlipVertical2 aliases
export { default as FlipVertical2Icon } from '../icons/flip-vertical-2';
// Flower2 aliases
export { default as Flower2Icon } from '../icons/flower-2';
// Flower aliases
export { default as FlowerIcon } from '../icons/flower';
// Focus aliases
export { default as FocusIcon } from '../icons/focus';
// FoldHorizontal aliases
export { default as FoldHorizontalIcon } from '../icons/fold-horizontal';
// FoldVertical aliases
export { default as FoldVerticalIcon } from '../icons/fold-vertical';
// FolderArchive aliases
export { default as FolderArchiveIcon } from '../icons/folder-archive';
// FolderCheck aliases
export { default as FolderCheckIcon } from '../icons/folder-check';
// FolderClock aliases
export { default as FolderClockIcon } from '../icons/folder-clock';
// FolderClosed aliases
export { default as FolderClosedIcon } from '../icons/folder-closed';
// FolderCode aliases
export { default as FolderCodeIcon } from '../icons/folder-code';
// FolderDot aliases
export { default as FolderDotIcon } from '../icons/folder-dot';
// FolderDown aliases
export { default as FolderDownIcon } from '../icons/folder-down';
// FolderGit2 aliases
export { default as FolderGit2Icon } from '../icons/folder-git-2';
// FolderGit aliases
export { default as FolderGitIcon } from '../icons/folder-git';
// FolderHeart aliases
export { default as FolderHeartIcon } from '../icons/folder-heart';
// FolderKanban aliases
export { default as FolderKanbanIcon } from '../icons/folder-kanban';
// FolderInput aliases
export { default as FolderInputIcon } from '../icons/folder-input';
// FolderKey aliases
export { default as FolderKeyIcon } from '../icons/folder-key';
// FolderLock aliases
export { default as FolderLockIcon } from '../icons/folder-lock';
// FolderMinus aliases
export { default as FolderMinusIcon } from '../icons/folder-minus';
// FolderOpenDot aliases
export { default as FolderOpenDotIcon } from '../icons/folder-open-dot';
// FolderOutput aliases
export { default as FolderOutputIcon } from '../icons/folder-output';
// FolderOpen aliases
export { default as FolderOpenIcon } from '../icons/folder-open';
// FolderPlus aliases
export { default as FolderPlusIcon } from '../icons/folder-plus';
// FolderRoot aliases
export { default as FolderRootIcon } from '../icons/folder-root';
// FolderSearch2 aliases
export { default as FolderSearch2Icon } from '../icons/folder-search-2';
// FolderSearch aliases
export { default as FolderSearchIcon } from '../icons/folder-search';
// FolderSymlink aliases
export { default as FolderSymlinkIcon } from '../icons/folder-symlink';
// FolderSync aliases
export { default as FolderSyncIcon } from '../icons/folder-sync';
// FolderTree aliases
export { default as FolderTreeIcon } from '../icons/folder-tree';
// FolderUp aliases
export { default as FolderUpIcon } from '../icons/folder-up';
// FolderX aliases
export { default as FolderXIcon } from '../icons/folder-x';
// Folder aliases
export { default as FolderIcon } from '../icons/folder';
// Footprints aliases
export { default as FootprintsIcon } from '../icons/footprints';
// Folders aliases
export { default as FoldersIcon } from '../icons/folders';
// Forklift aliases
export { default as ForkliftIcon } from '../icons/forklift';
// Forward aliases
export { default as ForwardIcon } from '../icons/forward';
// Framer aliases
export { default as FramerIcon } from '../icons/framer';
// Frame aliases
export { default as FrameIcon } from '../icons/frame';
// Frown aliases
export { default as FrownIcon } from '../icons/frown';
// Fuel aliases
export { default as FuelIcon } from '../icons/fuel';
// Fullscreen aliases
export { default as FullscreenIcon } from '../icons/fullscreen';
// FunnelPlus aliases
export { default as FunnelPlusIcon } from '../icons/funnel-plus';
// GalleryHorizontalEnd aliases
export { default as GalleryHorizontalEndIcon } from '../icons/gallery-horizontal-end';
// GalleryHorizontal aliases
export { default as GalleryHorizontalIcon } from '../icons/gallery-horizontal';
// GalleryThumbnails aliases
export { default as GalleryThumbnailsIcon } from '../icons/gallery-thumbnails';
// GalleryVertical aliases
export { default as GalleryVerticalIcon } from '../icons/gallery-vertical';
// GalleryVerticalEnd aliases
export { default as GalleryVerticalEndIcon } from '../icons/gallery-vertical-end';
// Gamepad2 aliases
export { default as Gamepad2Icon } from '../icons/gamepad-2';
// Gamepad aliases
export { default as GamepadIcon } from '../icons/gamepad';
// Gauge aliases
export { default as GaugeIcon } from '../icons/gauge';
// Gem aliases
export { default as GemIcon } from '../icons/gem';
// Gavel aliases
export { default as GavelIcon } from '../icons/gavel';
// GeorgianLari aliases
export { default as GeorgianLariIcon } from '../icons/georgian-lari';
// Ghost aliases
export { default as GhostIcon } from '../icons/ghost';
// GitBranchPlus aliases
export { default as GitBranchPlusIcon } from '../icons/git-branch-plus';
// Gift aliases
export { default as GiftIcon } from '../icons/gift';
// GitBranch aliases
export { default as GitBranchIcon } from '../icons/git-branch';
// GitCommitVertical aliases
export { default as GitCommitVerticalIcon } from '../icons/git-commit-vertical';
// GitCompareArrows aliases
export { default as GitCompareArrowsIcon } from '../icons/git-compare-arrows';
// GitCompare aliases
export { default as GitCompareIcon } from '../icons/git-compare';
// GitFork aliases
export { default as GitForkIcon } from '../icons/git-fork';
// GitMerge aliases
export { default as GitMergeIcon } from '../icons/git-merge';
// GitGraph aliases
export { default as GitGraphIcon } from '../icons/git-graph';
// GitPullRequestClosed aliases
export { default as GitPullRequestClosedIcon } from '../icons/git-pull-request-closed';
// GitPullRequestArrow aliases
export { default as GitPullRequestArrowIcon } from '../icons/git-pull-request-arrow';
// GitPullRequestCreateArrow aliases
export { default as GitPullRequestCreateArrowIcon } from '../icons/git-pull-request-create-arrow';
// GitPullRequestCreate aliases
export { default as GitPullRequestCreateIcon } from '../icons/git-pull-request-create';
// GitPullRequestDraft aliases
export { default as GitPullRequestDraftIcon } from '../icons/git-pull-request-draft';
// Github aliases
export { default as GithubIcon } from '../icons/github';
// GitPullRequest aliases
export { default as GitPullRequestIcon } from '../icons/git-pull-request';
// Gitlab aliases
export { default as GitlabIcon } from '../icons/gitlab';
// GlassWater aliases
export { default as GlassWaterIcon } from '../icons/glass-water';
// GlobeLock aliases
export { default as GlobeLockIcon } from '../icons/globe-lock';
// Glasses aliases
export { default as GlassesIcon } from '../icons/glasses';
// Globe aliases
export { default as GlobeIcon } from '../icons/globe';
// Goal aliases
export { default as GoalIcon } from '../icons/goal';
// Gpu aliases
export { default as GpuIcon } from '../icons/gpu';
// GraduationCap aliases
export { default as GraduationCapIcon } from '../icons/graduation-cap';
// Grape aliases
export { default as GrapeIcon } from '../icons/grape';
// Grid3x2 aliases
export { default as Grid3x2Icon } from '../icons/grid-3x2';
// GripHorizontal aliases
export { default as GripHorizontalIcon } from '../icons/grip-horizontal';
// GripVertical aliases
export { default as GripVerticalIcon } from '../icons/grip-vertical';
// Grip aliases
export { default as GripIcon } from '../icons/grip';
// Group aliases
export { default as GroupIcon } from '../icons/group';
// Guitar aliases
export { default as GuitarIcon } from '../icons/guitar';
// Ham aliases
export { default as HamIcon } from '../icons/ham';
// Hammer aliases
export { default as HammerIcon } from '../icons/hammer';
// Hamburger aliases
export { default as HamburgerIcon } from '../icons/hamburger';
// HandCoins aliases
export { default as HandCoinsIcon } from '../icons/hand-coins';
// HandFist aliases
export { default as HandFistIcon } from '../icons/hand-fist';
// HandHeart aliases
export { default as HandHeartIcon } from '../icons/hand-heart';
// HandMetal aliases
export { default as HandMetalIcon } from '../icons/hand-metal';
// HandPlatter aliases
export { default as HandPlatterIcon } from '../icons/hand-platter';
// Hand aliases
export { default as HandIcon } from '../icons/hand';
// Handbag aliases
export { default as HandbagIcon } from '../icons/handbag';
// HardDriveDownload aliases
export { default as HardDriveDownloadIcon } from '../icons/hard-drive-download';
// Handshake aliases
export { default as HandshakeIcon } from '../icons/handshake';
// HardDriveUpload aliases
export { default as HardDriveUploadIcon } from '../icons/hard-drive-upload';
// HardDrive aliases
export { default as HardDriveIcon } from '../icons/hard-drive';
// HardHat aliases
export { default as HardHatIcon } from '../icons/hard-hat';
// Hash aliases
export { default as HashIcon } from '../icons/hash';
// HatGlasses aliases
export { default as HatGlassesIcon } from '../icons/hat-glasses';
// Haze aliases
export { default as HazeIcon } from '../icons/haze';
// Heading1 aliases
export { default as Heading1Icon } from '../icons/heading-1';
// HdmiPort aliases
export { default as HdmiPortIcon } from '../icons/hdmi-port';
// Heading2 aliases
export { default as Heading2Icon } from '../icons/heading-2';
// Heading3 aliases
export { default as Heading3Icon } from '../icons/heading-3';
// Heading4 aliases
export { default as Heading4Icon } from '../icons/heading-4';
// Heading5 aliases
export { default as Heading5Icon } from '../icons/heading-5';
// Heading6 aliases
export { default as Heading6Icon } from '../icons/heading-6';
// Heading aliases
export { default as HeadingIcon } from '../icons/heading';
// HeadphoneOff aliases
export { default as HeadphoneOffIcon } from '../icons/headphone-off';
// Headphones aliases
export { default as HeadphonesIcon } from '../icons/headphones';
// Headset aliases
export { default as HeadsetIcon } from '../icons/headset';
// HeartCrack aliases
export { default as HeartCrackIcon } from '../icons/heart-crack';
// HeartHandshake aliases
export { default as HeartHandshakeIcon } from '../icons/heart-handshake';
// HeartOff aliases
export { default as HeartOffIcon } from '../icons/heart-off';
// HeartMinus aliases
export { default as HeartMinusIcon } from '../icons/heart-minus';
// HeartPlus aliases
export { default as HeartPlusIcon } from '../icons/heart-plus';
// HeartPulse aliases
export { default as HeartPulseIcon } from '../icons/heart-pulse';
// Heart aliases
export { default as HeartIcon } from '../icons/heart';
// Heater aliases
export { default as HeaterIcon } from '../icons/heater';
// Hexagon aliases
export { default as HexagonIcon } from '../icons/hexagon';
// Highlighter aliases
export { default as HighlighterIcon } from '../icons/highlighter';
// Hop aliases
export { default as HopIcon } from '../icons/hop';
// History aliases
export { default as HistoryIcon } from '../icons/history';
// HopOff aliases
export { default as HopOffIcon } from '../icons/hop-off';
// Hospital aliases
export { default as HospitalIcon } from '../icons/hospital';
// Hotel aliases
export { default as HotelIcon } from '../icons/hotel';
// Hourglass aliases
export { default as HourglassIcon } from '../icons/hourglass';
// HouseHeart aliases
export { default as HouseHeartIcon } from '../icons/house-heart';
// HousePlug aliases
export { default as HousePlugIcon } from '../icons/house-plug';
// HousePlus aliases
export { default as HousePlusIcon } from '../icons/house-plus';
// HouseWifi aliases
export { default as HouseWifiIcon } from '../icons/house-wifi';
// IdCardLanyard aliases
export { default as IdCardLanyardIcon } from '../icons/id-card-lanyard';
// ImageDown aliases
export { default as ImageDownIcon } from '../icons/image-down';
// IdCard aliases
export { default as IdCardIcon } from '../icons/id-card';
// ImageOff aliases
export { default as ImageOffIcon } from '../icons/image-off';
// ImageMinus aliases
export { default as ImageMinusIcon } from '../icons/image-minus';
// ImagePlay aliases
export { default as ImagePlayIcon } from '../icons/image-play';
// ImagePlus aliases
export { default as ImagePlusIcon } from '../icons/image-plus';
// ImageUp aliases
export { default as ImageUpIcon } from '../icons/image-up';
// Image aliases
export { default as ImageIcon } from '../icons/image';
// ImageUpscale aliases
export { default as ImageUpscaleIcon } from '../icons/image-upscale';
// Images aliases
export { default as ImagesIcon } from '../icons/images';
// Import aliases
export { default as ImportIcon } from '../icons/import';
// Inbox aliases
export { default as InboxIcon } from '../icons/inbox';
// IndianRupee aliases
export { default as IndianRupeeIcon } from '../icons/indian-rupee';
// Infinity aliases
export { default as InfinityIcon } from '../icons/infinity';
// Info aliases
export { default as InfoIcon } from '../icons/info';
// InspectionPanel aliases
export { default as InspectionPanelIcon } from '../icons/inspection-panel';
// Instagram aliases
export { default as InstagramIcon } from '../icons/instagram';
// Italic aliases
export { default as ItalicIcon } from '../icons/italic';
// IterationCcw aliases
export { default as IterationCcwIcon } from '../icons/iteration-ccw';
// IterationCw aliases
export { default as IterationCwIcon } from '../icons/iteration-cw';
// JapaneseYen aliases
export { default as JapaneseYenIcon } from '../icons/japanese-yen';
// Joystick aliases
export { default as JoystickIcon } from '../icons/joystick';
// Kanban aliases
export { default as KanbanIcon } from '../icons/kanban';
// Kayak aliases
export { default as KayakIcon } from '../icons/kayak';
// KeyRound aliases
export { default as KeyRoundIcon } from '../icons/key-round';
// KeySquare aliases
export { default as KeySquareIcon } from '../icons/key-square';
// KeyboardMusic aliases
export { default as KeyboardMusicIcon } from '../icons/keyboard-music';
// Key aliases
export { default as KeyIcon } from '../icons/key';
// KeyboardOff aliases
export { default as KeyboardOffIcon } from '../icons/keyboard-off';
// Keyboard aliases
export { default as KeyboardIcon } from '../icons/keyboard';
// LampDesk aliases
export { default as LampDeskIcon } from '../icons/lamp-desk';
// LampCeiling aliases
export { default as LampCeilingIcon } from '../icons/lamp-ceiling';
// LampWallDown aliases
export { default as LampWallDownIcon } from '../icons/lamp-wall-down';
// LampFloor aliases
export { default as LampFloorIcon } from '../icons/lamp-floor';
// LampWallUp aliases
export { default as LampWallUpIcon } from '../icons/lamp-wall-up';
// Lamp aliases
export { default as LampIcon } from '../icons/lamp';
// Landmark aliases
export { default as LandmarkIcon } from '../icons/landmark';
// LandPlot aliases
export { default as LandPlotIcon } from '../icons/land-plot';
// Languages aliases
export { default as LanguagesIcon } from '../icons/languages';
// LaptopMinimalCheck aliases
export { default as LaptopMinimalCheckIcon } from '../icons/laptop-minimal-check';
// Laptop aliases
export { default as LaptopIcon } from '../icons/laptop';
// LassoSelect aliases
export { default as LassoSelectIcon } from '../icons/lasso-select';
// Lasso aliases
export { default as LassoIcon } from '../icons/lasso';
// Laugh aliases
export { default as LaughIcon } from '../icons/laugh';
// Layers2 aliases
export { default as Layers2Icon } from '../icons/layers-2';
// LayoutDashboard aliases
export { default as LayoutDashboardIcon } from '../icons/layout-dashboard';
// LayoutGrid aliases
export { default as LayoutGridIcon } from '../icons/layout-grid';
// LayoutList aliases
export { default as LayoutListIcon } from '../icons/layout-list';
// LayoutTemplate aliases
export { default as LayoutTemplateIcon } from '../icons/layout-template';
// LayoutPanelLeft aliases
export { default as LayoutPanelLeftIcon } from '../icons/layout-panel-left';
// Leaf aliases
export { default as LeafIcon } from '../icons/leaf';
// LeafyGreen aliases
export { default as LeafyGreenIcon } from '../icons/leafy-green';
// LayoutPanelTop aliases
export { default as LayoutPanelTopIcon } from '../icons/layout-panel-top';
// Lectern aliases
export { default as LecternIcon } from '../icons/lectern';
// LibraryBig aliases
export { default as LibraryBigIcon } from '../icons/library-big';
// Library aliases
export { default as LibraryIcon } from '../icons/library';
// LifeBuoy aliases
export { default as LifeBuoyIcon } from '../icons/life-buoy';
// LightbulbOff aliases
export { default as LightbulbOffIcon } from '../icons/lightbulb-off';
// Ligature aliases
export { default as LigatureIcon } from '../icons/ligature';
// Lightbulb aliases
export { default as LightbulbIcon } from '../icons/lightbulb';
// LineSquiggle aliases
export { default as LineSquiggleIcon } from '../icons/line-squiggle';
// Link2Off aliases
export { default as Link2OffIcon } from '../icons/link-2-off';
// Link aliases
export { default as LinkIcon } from '../icons/link';
// Link2 aliases
export { default as Link2Icon } from '../icons/link-2';
// ListCheck aliases
export { default as ListCheckIcon } from '../icons/list-check';
// Linkedin aliases
export { default as LinkedinIcon } from '../icons/linkedin';
// ListChecks aliases
export { default as ListChecksIcon } from '../icons/list-checks';
// ListChevronsDownUp aliases
export { default as ListChevronsDownUpIcon } from '../icons/list-chevrons-down-up';
// ListCollapse aliases
export { default as ListCollapseIcon } from '../icons/list-collapse';
// ListEnd aliases
export { default as ListEndIcon } from '../icons/list-end';
// ListChevronsUpDown aliases
export { default as ListChevronsUpDownIcon } from '../icons/list-chevrons-up-down';
// ListFilterPlus aliases
export { default as ListFilterPlusIcon } from '../icons/list-filter-plus';
// ListFilter aliases
export { default as ListFilterIcon } from '../icons/list-filter';
// ListMinus aliases
export { default as ListMinusIcon } from '../icons/list-minus';
// ListMusic aliases
export { default as ListMusicIcon } from '../icons/list-music';
// ListPlus aliases
export { default as ListPlusIcon } from '../icons/list-plus';
// ListOrdered aliases
export { default as ListOrderedIcon } from '../icons/list-ordered';
// ListStart aliases
export { default as ListStartIcon } from '../icons/list-start';
// ListRestart aliases
export { default as ListRestartIcon } from '../icons/list-restart';
// ListVideo aliases
export { default as ListVideoIcon } from '../icons/list-video';
// ListTodo aliases
export { default as ListTodoIcon } from '../icons/list-todo';
// ListTree aliases
export { default as ListTreeIcon } from '../icons/list-tree';
// ListX aliases
export { default as ListXIcon } from '../icons/list-x';
// List aliases
export { default as ListIcon } from '../icons/list';
// LoaderPinwheel aliases
export { default as LoaderPinwheelIcon } from '../icons/loader-pinwheel';
// Loader aliases
export { default as LoaderIcon } from '../icons/loader';
// LocateFixed aliases
export { default as LocateFixedIcon } from '../icons/locate-fixed';
// LocateOff aliases
export { default as LocateOffIcon } from '../icons/locate-off';
// LockKeyhole aliases
export { default as LockKeyholeIcon } from '../icons/lock-keyhole';
// Lock aliases
export { default as LockIcon } from '../icons/lock';
// Locate aliases
export { default as LocateIcon } from '../icons/locate';
// LogIn aliases
export { default as LogInIcon } from '../icons/log-in';
// LogOut aliases
export { default as LogOutIcon } from '../icons/log-out';
// Logs aliases
export { default as LogsIcon } from '../icons/logs';
// Luggage aliases
export { default as LuggageIcon } from '../icons/luggage';
// Lollipop aliases
export { default as LollipopIcon } from '../icons/lollipop';
// Magnet aliases
export { default as MagnetIcon } from '../icons/magnet';
// MailCheck aliases
export { default as MailCheckIcon } from '../icons/mail-check';
// MailMinus aliases
export { default as MailMinusIcon } from '../icons/mail-minus';
// MailOpen aliases
export { default as MailOpenIcon } from '../icons/mail-open';
// MailPlus aliases
export { default as MailPlusIcon } from '../icons/mail-plus';
// MailWarning aliases
export { default as MailWarningIcon } from '../icons/mail-warning';
// MailSearch aliases
export { default as MailSearchIcon } from '../icons/mail-search';
// MailX aliases
export { default as MailXIcon } from '../icons/mail-x';
// Mailbox aliases
export { default as MailboxIcon } from '../icons/mailbox';
// Mail aliases
export { default as MailIcon } from '../icons/mail';
// Mails aliases
export { default as MailsIcon } from '../icons/mails';
// MapMinus aliases
export { default as MapMinusIcon } from '../icons/map-minus';
// MapPinCheckInside aliases
export { default as MapPinCheckInsideIcon } from '../icons/map-pin-check-inside';
// MapPinCheck aliases
export { default as MapPinCheckIcon } from '../icons/map-pin-check';
// MapPinHouse aliases
export { default as MapPinHouseIcon } from '../icons/map-pin-house';
// MapPinMinusInside aliases
export { default as MapPinMinusInsideIcon } from '../icons/map-pin-minus-inside';
// MapPinMinus aliases
export { default as MapPinMinusIcon } from '../icons/map-pin-minus';
// MapPinPlusInside aliases
export { default as MapPinPlusInsideIcon } from '../icons/map-pin-plus-inside';
// MapPinOff aliases
export { default as MapPinOffIcon } from '../icons/map-pin-off';
// MapPinPlus aliases
export { default as MapPinPlusIcon } from '../icons/map-pin-plus';
// MapPinX aliases
export { default as MapPinXIcon } from '../icons/map-pin-x';
// MapPinXInside aliases
export { default as MapPinXInsideIcon } from '../icons/map-pin-x-inside';
// MapPin aliases
export { default as MapPinIcon } from '../icons/map-pin';
// MapPinned aliases
export { default as MapPinnedIcon } from '../icons/map-pinned';
// MapPlus aliases
export { default as MapPlusIcon } from '../icons/map-plus';
// Map aliases
export { default as MapIcon } from '../icons/map';
// MarsStroke aliases
export { default as MarsStrokeIcon } from '../icons/mars-stroke';
// Martini aliases
export { default as MartiniIcon } from '../icons/martini';
// Maximize2 aliases
export { default as Maximize2Icon } from '../icons/maximize-2';
// Maximize aliases
export { default as MaximizeIcon } from '../icons/maximize';
// Mars aliases
export { default as MarsIcon } from '../icons/mars';
// MegaphoneOff aliases
export { default as MegaphoneOffIcon } from '../icons/megaphone-off';
// Medal aliases
export { default as MedalIcon } from '../icons/medal';
// Megaphone aliases
export { default as MegaphoneIcon } from '../icons/megaphone';
// Meh aliases
export { default as MehIcon } from '../icons/meh';
// MemoryStick aliases
export { default as MemoryStickIcon } from '../icons/memory-stick';
// Menu aliases
export { default as MenuIcon } from '../icons/menu';
// Merge aliases
export { default as MergeIcon } from '../icons/merge';
// MessageCircleCode aliases
export { default as MessageCircleCodeIcon } from '../icons/message-circle-code';
// MessageCircleDashed aliases
export { default as MessageCircleDashedIcon } from '../icons/message-circle-dashed';
// MessageCircleHeart aliases
export { default as MessageCircleHeartIcon } from '../icons/message-circle-heart';
// MessageCircleMore aliases
export { default as MessageCircleMoreIcon } from '../icons/message-circle-more';
// MessageCirclePlus aliases
export { default as MessageCirclePlusIcon } from '../icons/message-circle-plus';
// MessageCircleReply aliases
export { default as MessageCircleReplyIcon } from '../icons/message-circle-reply';
// MessageCircleOff aliases
export { default as MessageCircleOffIcon } from '../icons/message-circle-off';
// MessageCircleWarning aliases
export { default as MessageCircleWarningIcon } from '../icons/message-circle-warning';
// MessageCircleX aliases
export { default as MessageCircleXIcon } from '../icons/message-circle-x';
// MessageCircle aliases
export { default as MessageCircleIcon } from '../icons/message-circle';
// MessageSquareCode aliases
export { default as MessageSquareCodeIcon } from '../icons/message-square-code';
// MessageSquareDiff aliases
export { default as MessageSquareDiffIcon } from '../icons/message-square-diff';
// MessageSquareDot aliases
export { default as MessageSquareDotIcon } from '../icons/message-square-dot';
// MessageSquareDashed aliases
export { default as MessageSquareDashedIcon } from '../icons/message-square-dashed';
// MessageSquareHeart aliases
export { default as MessageSquareHeartIcon } from '../icons/message-square-heart';
// MessageSquareMore aliases
export { default as MessageSquareMoreIcon } from '../icons/message-square-more';
// MessageSquarePlus aliases
export { default as MessageSquarePlusIcon } from '../icons/message-square-plus';
// MessageSquareOff aliases
export { default as MessageSquareOffIcon } from '../icons/message-square-off';
// MessageSquareLock aliases
export { default as MessageSquareLockIcon } from '../icons/message-square-lock';
// MessageSquareQuote aliases
export { default as MessageSquareQuoteIcon } from '../icons/message-square-quote';
// MessageSquareShare aliases
export { default as MessageSquareShareIcon } from '../icons/message-square-share';
// MessageSquareReply aliases
export { default as MessageSquareReplyIcon } from '../icons/message-square-reply';
// MessageSquareText aliases
export { default as MessageSquareTextIcon } from '../icons/message-square-text';
// MessageSquareWarning aliases
export { default as MessageSquareWarningIcon } from '../icons/message-square-warning';
// MessageSquareX aliases
export { default as MessageSquareXIcon } from '../icons/message-square-x';
// MessageSquare aliases
export { default as MessageSquareIcon } from '../icons/message-square';
// MessagesSquare aliases
export { default as MessagesSquareIcon } from '../icons/messages-square';
// Microchip aliases
export { default as MicrochipIcon } from '../icons/microchip';
// MicOff aliases
export { default as MicOffIcon } from '../icons/mic-off';
// Mic aliases
export { default as MicIcon } from '../icons/mic';
// Microscope aliases
export { default as MicroscopeIcon } from '../icons/microscope';
// Microwave aliases
export { default as MicrowaveIcon } from '../icons/microwave';
// MilkOff aliases
export { default as MilkOffIcon } from '../icons/milk-off';
// Milestone aliases
export { default as MilestoneIcon } from '../icons/milestone';
// Milk aliases
export { default as MilkIcon } from '../icons/milk';
// Minimize2 aliases
export { default as Minimize2Icon } from '../icons/minimize-2';
// Minimize aliases
export { default as MinimizeIcon } from '../icons/minimize';
// Minus aliases
export { default as MinusIcon } from '../icons/minus';
// MonitorCog aliases
export { default as MonitorCogIcon } from '../icons/monitor-cog';
// MonitorCheck aliases
export { default as MonitorCheckIcon } from '../icons/monitor-check';
// MonitorDown aliases
export { default as MonitorDownIcon } from '../icons/monitor-down';
// MonitorDot aliases
export { default as MonitorDotIcon } from '../icons/monitor-dot';
// MonitorPause aliases
export { default as MonitorPauseIcon } from '../icons/monitor-pause';
// MonitorOff aliases
export { default as MonitorOffIcon } from '../icons/monitor-off';
// MonitorPlay aliases
export { default as MonitorPlayIcon } from '../icons/monitor-play';
// MonitorSmartphone aliases
export { default as MonitorSmartphoneIcon } from '../icons/monitor-smartphone';
// MonitorSpeaker aliases
export { default as MonitorSpeakerIcon } from '../icons/monitor-speaker';
// MonitorUp aliases
export { default as MonitorUpIcon } from '../icons/monitor-up';
// MonitorX aliases
export { default as MonitorXIcon } from '../icons/monitor-x';
// MonitorStop aliases
export { default as MonitorStopIcon } from '../icons/monitor-stop';
// Monitor aliases
export { default as MonitorIcon } from '../icons/monitor';
// Moon aliases
export { default as MoonIcon } from '../icons/moon';
// MountainSnow aliases
export { default as MountainSnowIcon } from '../icons/mountain-snow';
// MoonStar aliases
export { default as MoonStarIcon } from '../icons/moon-star';
// Mountain aliases
export { default as MountainIcon } from '../icons/mountain';
// MouseOff aliases
export { default as MouseOffIcon } from '../icons/mouse-off';
// MousePointer2 aliases
export { default as MousePointer2Icon } from '../icons/mouse-pointer-2';
// MousePointer aliases
export { default as MousePointerIcon } from '../icons/mouse-pointer';
// MousePointerBan aliases
export { default as MousePointerBanIcon } from '../icons/mouse-pointer-ban';
// MousePointerClick aliases
export { default as MousePointerClickIcon } from '../icons/mouse-pointer-click';
// MoveDiagonal2 aliases
export { default as MoveDiagonal2Icon } from '../icons/move-diagonal-2';
// Mouse aliases
export { default as MouseIcon } from '../icons/mouse';
// MoveDownLeft aliases
export { default as MoveDownLeftIcon } from '../icons/move-down-left';
// MoveDiagonal aliases
export { default as MoveDiagonalIcon } from '../icons/move-diagonal';
// MoveDownRight aliases
export { default as MoveDownRightIcon } from '../icons/move-down-right';
// MoveHorizontal aliases
export { default as MoveHorizontalIcon } from '../icons/move-horizontal';
// MoveDown aliases
export { default as MoveDownIcon } from '../icons/move-down';
// MoveLeft aliases
export { default as MoveLeftIcon } from '../icons/move-left';
// MoveRight aliases
export { default as MoveRightIcon } from '../icons/move-right';
// MoveUpLeft aliases
export { default as MoveUpLeftIcon } from '../icons/move-up-left';
// MoveUpRight aliases
export { default as MoveUpRightIcon } from '../icons/move-up-right';
// MoveVertical aliases
export { default as MoveVerticalIcon } from '../icons/move-vertical';
// MoveUp aliases
export { default as MoveUpIcon } from '../icons/move-up';
// Move aliases
export { default as MoveIcon } from '../icons/move';
// Music2 aliases
export { default as Music2Icon } from '../icons/music-2';
// Music3 aliases
export { default as Music3Icon } from '../icons/music-3';
// Music4 aliases
export { default as Music4Icon } from '../icons/music-4';
// Navigation2Off aliases
export { default as Navigation2OffIcon } from '../icons/navigation-2-off';
// Music aliases
export { default as MusicIcon } from '../icons/music';
// NavigationOff aliases
export { default as NavigationOffIcon } from '../icons/navigation-off';
// Navigation aliases
export { default as NavigationIcon } from '../icons/navigation';
// Navigation2 aliases
export { default as Navigation2Icon } from '../icons/navigation-2';
// Network aliases
export { default as NetworkIcon } from '../icons/network';
// Newspaper aliases
export { default as NewspaperIcon } from '../icons/newspaper';
// Nfc aliases
export { default as NfcIcon } from '../icons/nfc';
// NonBinary aliases
export { default as NonBinaryIcon } from '../icons/non-binary';
// NotebookPen aliases
export { default as NotebookPenIcon } from '../icons/notebook-pen';
// NotebookTabs aliases
export { default as NotebookTabsIcon } from '../icons/notebook-tabs';
// NotebookText aliases
export { default as NotebookTextIcon } from '../icons/notebook-text';
// Notebook aliases
export { default as NotebookIcon } from '../icons/notebook';
// NotepadTextDashed aliases
export { default as NotepadTextDashedIcon } from '../icons/notepad-text-dashed';
// NutOff aliases
export { default as NutOffIcon } from '../icons/nut-off';
// NotepadText aliases
export { default as NotepadTextIcon } from '../icons/notepad-text';
// Nut aliases
export { default as NutIcon } from '../icons/nut';
// OctagonMinus aliases
export { default as OctagonMinusIcon } from '../icons/octagon-minus';
// Octagon aliases
export { default as OctagonIcon } from '../icons/octagon';
// Omega aliases
export { default as OmegaIcon } from '../icons/omega';
// Option aliases
export { default as OptionIcon } from '../icons/option';
// Orbit aliases
export { default as OrbitIcon } from '../icons/orbit';
// Origami aliases
export { default as OrigamiIcon } from '../icons/origami';
// Package2 aliases
export { default as Package2Icon } from '../icons/package-2';
// PackageCheck aliases
export { default as PackageCheckIcon } from '../icons/package-check';
// PackageMinus aliases
export { default as PackageMinusIcon } from '../icons/package-minus';
// PackageOpen aliases
export { default as PackageOpenIcon } from '../icons/package-open';
// PackagePlus aliases
export { default as PackagePlusIcon } from '../icons/package-plus';
// PackageSearch aliases
export { default as PackageSearchIcon } from '../icons/package-search';
// PackageX aliases
export { default as PackageXIcon } from '../icons/package-x';
// Package aliases
export { default as PackageIcon } from '../icons/package';
// PaintBucket aliases
export { default as PaintBucketIcon } from '../icons/paint-bucket';
// PaintRoller aliases
export { default as PaintRollerIcon } from '../icons/paint-roller';
// Palette aliases
export { default as PaletteIcon } from '../icons/palette';
// Paintbrush aliases
export { default as PaintbrushIcon } from '../icons/paintbrush';
// Panda aliases
export { default as PandaIcon } from '../icons/panda';
// PanelBottomClose aliases
export { default as PanelBottomCloseIcon } from '../icons/panel-bottom-close';
// PanelBottomOpen aliases
export { default as PanelBottomOpenIcon } from '../icons/panel-bottom-open';
// PanelBottom aliases
export { default as PanelBottomIcon } from '../icons/panel-bottom';
// PanelLeftRightDashed aliases
export { default as PanelLeftRightDashedIcon } from '../icons/panel-left-right-dashed';
// PanelRightClose aliases
export { default as PanelRightCloseIcon } from '../icons/panel-right-close';
// PanelRightOpen aliases
export { default as PanelRightOpenIcon } from '../icons/panel-right-open';
// PanelRight aliases
export { default as PanelRightIcon } from '../icons/panel-right';
// PanelTopBottomDashed aliases
export { default as PanelTopBottomDashedIcon } from '../icons/panel-top-bottom-dashed';
// PanelTopClose aliases
export { default as PanelTopCloseIcon } from '../icons/panel-top-close';
// PanelTop aliases
export { default as PanelTopIcon } from '../icons/panel-top';
// PanelTopOpen aliases
export { default as PanelTopOpenIcon } from '../icons/panel-top-open';
// PanelsLeftBottom aliases
export { default as PanelsLeftBottomIcon } from '../icons/panels-left-bottom';
// PanelsRightBottom aliases
export { default as PanelsRightBottomIcon } from '../icons/panels-right-bottom';
// Paperclip aliases
export { default as PaperclipIcon } from '../icons/paperclip';
// Parentheses aliases
export { default as ParenthesesIcon } from '../icons/parentheses';
// PartyPopper aliases
export { default as PartyPopperIcon } from '../icons/party-popper';
// Pause aliases
export { default as PauseIcon } from '../icons/pause';
// ParkingMeter aliases
export { default as ParkingMeterIcon } from '../icons/parking-meter';
// PawPrint aliases
export { default as PawPrintIcon } from '../icons/paw-print';
// PcCase aliases
export { default as PcCaseIcon } from '../icons/pc-case';
// PenOff aliases
export { default as PenOffIcon } from '../icons/pen-off';
// PenTool aliases
export { default as PenToolIcon } from '../icons/pen-tool';
// PencilLine aliases
export { default as PencilLineIcon } from '../icons/pencil-line';
// PencilOff aliases
export { default as PencilOffIcon } from '../icons/pencil-off';
// PencilRuler aliases
export { default as PencilRulerIcon } from '../icons/pencil-ruler';
// Pentagon aliases
export { default as PentagonIcon } from '../icons/pentagon';
// Percent aliases
export { default as PercentIcon } from '../icons/percent';
// Pencil aliases
export { default as PencilIcon } from '../icons/pencil';
// PersonStanding aliases
export { default as PersonStandingIcon } from '../icons/person-standing';
// PhilippinePeso aliases
export { default as PhilippinePesoIcon } from '../icons/philippine-peso';
// PhoneCall aliases
export { default as PhoneCallIcon } from '../icons/phone-call';
// PhoneForwarded aliases
export { default as PhoneForwardedIcon } from '../icons/phone-forwarded';
// PhoneIncoming aliases
export { default as PhoneIncomingIcon } from '../icons/phone-incoming';
// PhoneMissed aliases
export { default as PhoneMissedIcon } from '../icons/phone-missed';
// PhoneOff aliases
export { default as PhoneOffIcon } from '../icons/phone-off';
// PhoneOutgoing aliases
export { default as PhoneOutgoingIcon } from '../icons/phone-outgoing';
// Phone aliases
export { default as PhoneIcon } from '../icons/phone';
// Pi aliases
export { default as PiIcon } from '../icons/pi';
// Piano aliases
export { default as PianoIcon } from '../icons/piano';
// Pickaxe aliases
export { default as PickaxeIcon } from '../icons/pickaxe';
// PictureInPicture aliases
export { default as PictureInPictureIcon } from '../icons/picture-in-picture';
// PictureInPicture2 aliases
export { default as PictureInPicture2Icon } from '../icons/picture-in-picture-2';
// PiggyBank aliases
export { default as PiggyBankIcon } from '../icons/piggy-bank';
// PilcrowRight aliases
export { default as PilcrowRightIcon } from '../icons/pilcrow-right';
// PilcrowLeft aliases
export { default as PilcrowLeftIcon } from '../icons/pilcrow-left';
// Pilcrow aliases
export { default as PilcrowIcon } from '../icons/pilcrow';
// PillBottle aliases
export { default as PillBottleIcon } from '../icons/pill-bottle';
// Pill aliases
export { default as PillIcon } from '../icons/pill';
// PinOff aliases
export { default as PinOffIcon } from '../icons/pin-off';
// Pizza aliases
export { default as PizzaIcon } from '../icons/pizza';
// Pipette aliases
export { default as PipetteIcon } from '../icons/pipette';
// Pin aliases
export { default as PinIcon } from '../icons/pin';
// PlaneLanding aliases
export { default as PlaneLandingIcon } from '../icons/plane-landing';
// PlaneTakeoff aliases
export { default as PlaneTakeoffIcon } from '../icons/plane-takeoff';
// Play aliases
export { default as PlayIcon } from '../icons/play';
// Plane aliases
export { default as PlaneIcon } from '../icons/plane';
// Plug2 aliases
export { default as Plug2Icon } from '../icons/plug-2';
// Plug aliases
export { default as PlugIcon } from '../icons/plug';
// PocketKnife aliases
export { default as PocketKnifeIcon } from '../icons/pocket-knife';
// Plus aliases
export { default as PlusIcon } from '../icons/plus';
// Pocket aliases
export { default as PocketIcon } from '../icons/pocket';
// Podcast aliases
export { default as PodcastIcon } from '../icons/podcast';
// Pointer aliases
export { default as PointerIcon } from '../icons/pointer';
// PointerOff aliases
export { default as PointerOffIcon } from '../icons/pointer-off';
// Popsicle aliases
export { default as PopsicleIcon } from '../icons/popsicle';
// Popcorn aliases
export { default as PopcornIcon } from '../icons/popcorn';
// PowerOff aliases
export { default as PowerOffIcon } from '../icons/power-off';
// PoundSterling aliases
export { default as PoundSterlingIcon } from '../icons/pound-sterling';
// Power aliases
export { default as PowerIcon } from '../icons/power';
// Presentation aliases
export { default as PresentationIcon } from '../icons/presentation';
// Printer aliases
export { default as PrinterIcon } from '../icons/printer';
// Projector aliases
export { default as ProjectorIcon } from '../icons/projector';
// PrinterCheck aliases
export { default as PrinterCheckIcon } from '../icons/printer-check';
// Puzzle aliases
export { default as PuzzleIcon } from '../icons/puzzle';
// Proportions aliases
export { default as ProportionsIcon } from '../icons/proportions';
// QrCode aliases
export { default as QrCodeIcon } from '../icons/qr-code';
// Pyramid aliases
export { default as PyramidIcon } from '../icons/pyramid';
// Quote aliases
export { default as QuoteIcon } from '../icons/quote';
// Radar aliases
export { default as RadarIcon } from '../icons/radar';
// Rabbit aliases
export { default as RabbitIcon } from '../icons/rabbit';
// Radiation aliases
export { default as RadiationIcon } from '../icons/radiation';
// Radical aliases
export { default as RadicalIcon } from '../icons/radical';
// RadioReceiver aliases
export { default as RadioReceiverIcon } from '../icons/radio-receiver';
// RadioTower aliases
export { default as RadioTowerIcon } from '../icons/radio-tower';
// Radius aliases
export { default as RadiusIcon } from '../icons/radius';
// RailSymbol aliases
export { default as RailSymbolIcon } from '../icons/rail-symbol';
// Radio aliases
export { default as RadioIcon } from '../icons/radio';
// Rat aliases
export { default as RatIcon } from '../icons/rat';
// Rainbow aliases
export { default as RainbowIcon } from '../icons/rainbow';
// Ratio aliases
export { default as RatioIcon } from '../icons/ratio';
// ReceiptEuro aliases
export { default as ReceiptEuroIcon } from '../icons/receipt-euro';
// ReceiptCent aliases
export { default as ReceiptCentIcon } from '../icons/receipt-cent';
// ReceiptIndianRupee aliases
export { default as ReceiptIndianRupeeIcon } from '../icons/receipt-indian-rupee';
// ReceiptJapaneseYen aliases
export { default as ReceiptJapaneseYenIcon } from '../icons/receipt-japanese-yen';
// ReceiptPoundSterling aliases
export { default as ReceiptPoundSterlingIcon } from '../icons/receipt-pound-sterling';
// ReceiptRussianRuble aliases
export { default as ReceiptRussianRubleIcon } from '../icons/receipt-russian-ruble';
// ReceiptSwissFranc aliases
export { default as ReceiptSwissFrancIcon } from '../icons/receipt-swiss-franc';
// ReceiptText aliases
export { default as ReceiptTextIcon } from '../icons/receipt-text';
// Receipt aliases
export { default as ReceiptIcon } from '../icons/receipt';
// ReceiptTurkishLira aliases
export { default as ReceiptTurkishLiraIcon } from '../icons/receipt-turkish-lira';
// RectangleHorizontal aliases
export { default as RectangleHorizontalIcon } from '../icons/rectangle-horizontal';
// RectangleCircle aliases
export { default as RectangleCircleIcon } from '../icons/rectangle-circle';
// RectangleGoggles aliases
export { default as RectangleGogglesIcon } from '../icons/rectangle-goggles';
// RectangleVertical aliases
export { default as RectangleVerticalIcon } from '../icons/rectangle-vertical';
// Recycle aliases
export { default as RecycleIcon } from '../icons/recycle';
// RedoDot aliases
export { default as RedoDotIcon } from '../icons/redo-dot';
// Redo2 aliases
export { default as Redo2Icon } from '../icons/redo-2';
// Redo aliases
export { default as RedoIcon } from '../icons/redo';
// RefreshCcwDot aliases
export { default as RefreshCcwDotIcon } from '../icons/refresh-ccw-dot';
// RefreshCcw aliases
export { default as RefreshCcwIcon } from '../icons/refresh-ccw';
// RefreshCw aliases
export { default as RefreshCwIcon } from '../icons/refresh-cw';
// RefreshCwOff aliases
export { default as RefreshCwOffIcon } from '../icons/refresh-cw-off';
// Refrigerator aliases
export { default as RefrigeratorIcon } from '../icons/refrigerator';
// Regex aliases
export { default as RegexIcon } from '../icons/regex';
// RemoveFormatting aliases
export { default as RemoveFormattingIcon } from '../icons/remove-formatting';
// Repeat1 aliases
export { default as Repeat1Icon } from '../icons/repeat-1';
// Repeat2 aliases
export { default as Repeat2Icon } from '../icons/repeat-2';
// Repeat aliases
export { default as RepeatIcon } from '../icons/repeat';
// Replace aliases
export { default as ReplaceIcon } from '../icons/replace';
// ReplaceAll aliases
export { default as ReplaceAllIcon } from '../icons/replace-all';
// ReplyAll aliases
export { default as ReplyAllIcon } from '../icons/reply-all';
// Reply aliases
export { default as ReplyIcon } from '../icons/reply';
// Rewind aliases
export { default as RewindIcon } from '../icons/rewind';
// Rocket aliases
export { default as RocketIcon } from '../icons/rocket';
// Ribbon aliases
export { default as RibbonIcon } from '../icons/ribbon';
// RockingChair aliases
export { default as RockingChairIcon } from '../icons/rocking-chair';
// Rose aliases
export { default as RoseIcon } from '../icons/rose';
// RollerCoaster aliases
export { default as RollerCoasterIcon } from '../icons/roller-coaster';
// RotateCcwSquare aliases
export { default as RotateCcwSquareIcon } from '../icons/rotate-ccw-square';
// RotateCcwKey aliases
export { default as RotateCcwKeyIcon } from '../icons/rotate-ccw-key';
// RotateCcw aliases
export { default as RotateCcwIcon } from '../icons/rotate-ccw';
// RotateCwSquare aliases
export { default as RotateCwSquareIcon } from '../icons/rotate-cw-square';
// RotateCw aliases
export { default as RotateCwIcon } from '../icons/rotate-cw';
// Route aliases
export { default as RouteIcon } from '../icons/route';
// Router aliases
export { default as RouterIcon } from '../icons/router';
// Rows4 aliases
export { default as Rows4Icon } from '../icons/rows-4';
// RouteOff aliases
export { default as RouteOffIcon } from '../icons/route-off';
// Rss aliases
export { default as RssIcon } from '../icons/rss';
// RulerDimensionLine aliases
export { default as RulerDimensionLineIcon } from '../icons/ruler-dimension-line';
// Ruler aliases
export { default as RulerIcon } from '../icons/ruler';
// RussianRuble aliases
export { default as RussianRubleIcon } from '../icons/russian-ruble';
// Sailboat aliases
export { default as SailboatIcon } from '../icons/sailboat';
// Salad aliases
export { default as SaladIcon } from '../icons/salad';
// Satellite aliases
export { default as SatelliteIcon } from '../icons/satellite';
// Sandwich aliases
export { default as SandwichIcon } from '../icons/sandwich';
// SatelliteDish aliases
export { default as SatelliteDishIcon } from '../icons/satellite-dish';
// SaudiRiyal aliases
export { default as SaudiRiyalIcon } from '../icons/saudi-riyal';
// SaveAll aliases
export { default as SaveAllIcon } from '../icons/save-all';
// SaveOff aliases
export { default as SaveOffIcon } from '../icons/save-off';
// Scale aliases
export { default as ScaleIcon } from '../icons/scale';
// Save aliases
export { default as SaveIcon } from '../icons/save';
// Scaling aliases
export { default as ScalingIcon } from '../icons/scaling';
// ScanBarcode aliases
export { default as ScanBarcodeIcon } from '../icons/scan-barcode';
// ScanEye aliases
export { default as ScanEyeIcon } from '../icons/scan-eye';
// ScanFace aliases
export { default as ScanFaceIcon } from '../icons/scan-face';
// ScanHeart aliases
export { default as ScanHeartIcon } from '../icons/scan-heart';
// ScanLine aliases
export { default as ScanLineIcon } from '../icons/scan-line';
// ScanSearch aliases
export { default as ScanSearchIcon } from '../icons/scan-search';
// ScanQrCode aliases
export { default as ScanQrCodeIcon } from '../icons/scan-qr-code';
// ScanText aliases
export { default as ScanTextIcon } from '../icons/scan-text';
// Scan aliases
export { default as ScanIcon } from '../icons/scan';
// School aliases
export { default as SchoolIcon } from '../icons/school';
// ScissorsLineDashed aliases
export { default as ScissorsLineDashedIcon } from '../icons/scissors-line-dashed';
// ScreenShareOff aliases
export { default as ScreenShareOffIcon } from '../icons/screen-share-off';
// ScreenShare aliases
export { default as ScreenShareIcon } from '../icons/screen-share';
// Scissors aliases
export { default as ScissorsIcon } from '../icons/scissors';
// ScrollText aliases
export { default as ScrollTextIcon } from '../icons/scroll-text';
// Scroll aliases
export { default as ScrollIcon } from '../icons/scroll';
// SearchCheck aliases
export { default as SearchCheckIcon } from '../icons/search-check';
// SearchCode aliases
export { default as SearchCodeIcon } from '../icons/search-code';
// SearchX aliases
export { default as SearchXIcon } from '../icons/search-x';
// SearchSlash aliases
export { default as SearchSlashIcon } from '../icons/search-slash';
// Search aliases
export { default as SearchIcon } from '../icons/search';
// Section aliases
export { default as SectionIcon } from '../icons/section';
// Send aliases
export { default as SendIcon } from '../icons/send';
// SendToBack aliases
export { default as SendToBackIcon } from '../icons/send-to-back';
// SeparatorHorizontal aliases
export { default as SeparatorHorizontalIcon } from '../icons/separator-horizontal';
// ServerCog aliases
export { default as ServerCogIcon } from '../icons/server-cog';
// SeparatorVertical aliases
export { default as SeparatorVerticalIcon } from '../icons/separator-vertical';
// ServerCrash aliases
export { default as ServerCrashIcon } from '../icons/server-crash';
// ServerOff aliases
export { default as ServerOffIcon } from '../icons/server-off';
// Server aliases
export { default as ServerIcon } from '../icons/server';
// Settings2 aliases
export { default as Settings2Icon } from '../icons/settings-2';
// Settings aliases
export { default as SettingsIcon } from '../icons/settings';
// Shapes aliases
export { default as ShapesIcon } from '../icons/shapes';
// Share2 aliases
export { default as Share2Icon } from '../icons/share-2';
// Sheet aliases
export { default as SheetIcon } from '../icons/sheet';
// Share aliases
export { default as ShareIcon } from '../icons/share';
// Shell aliases
export { default as ShellIcon } from '../icons/shell';
// ShieldAlert aliases
export { default as ShieldAlertIcon } from '../icons/shield-alert';
// ShieldBan aliases
export { default as ShieldBanIcon } from '../icons/shield-ban';
// ShieldEllipsis aliases
export { default as ShieldEllipsisIcon } from '../icons/shield-ellipsis';
// ShieldCheck aliases
export { default as ShieldCheckIcon } from '../icons/shield-check';
// ShieldMinus aliases
export { default as ShieldMinusIcon } from '../icons/shield-minus';
// ShieldHalf aliases
export { default as ShieldHalfIcon } from '../icons/shield-half';
// ShieldOff aliases
export { default as ShieldOffIcon } from '../icons/shield-off';
// ShieldPlus aliases
export { default as ShieldPlusIcon } from '../icons/shield-plus';
// Shield aliases
export { default as ShieldIcon } from '../icons/shield';
// ShieldUser aliases
export { default as ShieldUserIcon } from '../icons/shield-user';
// ShipWheel aliases
export { default as ShipWheelIcon } from '../icons/ship-wheel';
// Ship aliases
export { default as ShipIcon } from '../icons/ship';
// Shirt aliases
export { default as ShirtIcon } from '../icons/shirt';
// ShoppingBag aliases
export { default as ShoppingBagIcon } from '../icons/shopping-bag';
// ShoppingBasket aliases
export { default as ShoppingBasketIcon } from '../icons/shopping-basket';
// ShoppingCart aliases
export { default as ShoppingCartIcon } from '../icons/shopping-cart';
// Shovel aliases
export { default as ShovelIcon } from '../icons/shovel';
// ShowerHead aliases
export { default as ShowerHeadIcon } from '../icons/shower-head';
// Shredder aliases
export { default as ShredderIcon } from '../icons/shredder';
// Shrimp aliases
export { default as ShrimpIcon } from '../icons/shrimp';
// Shrink aliases
export { default as ShrinkIcon } from '../icons/shrink';
// Shuffle aliases
export { default as ShuffleIcon } from '../icons/shuffle';
// Shrub aliases
export { default as ShrubIcon } from '../icons/shrub';
// Sigma aliases
export { default as SigmaIcon } from '../icons/sigma';
// SignalHigh aliases
export { default as SignalHighIcon } from '../icons/signal-high';
// SignalLow aliases
export { default as SignalLowIcon } from '../icons/signal-low';
// SignalMedium aliases
export { default as SignalMediumIcon } from '../icons/signal-medium';
// SignalZero aliases
export { default as SignalZeroIcon } from '../icons/signal-zero';
// Signal aliases
export { default as SignalIcon } from '../icons/signal';
// Signature aliases
export { default as SignatureIcon } from '../icons/signature';
// Siren aliases
export { default as SirenIcon } from '../icons/siren';
// SignpostBig aliases
export { default as SignpostBigIcon } from '../icons/signpost-big';
// Signpost aliases
export { default as SignpostIcon } from '../icons/signpost';
// SkipBack aliases
export { default as SkipBackIcon } from '../icons/skip-back';
// SkipForward aliases
export { default as SkipForwardIcon } from '../icons/skip-forward';
// Skull aliases
export { default as SkullIcon } from '../icons/skull';
// Slash aliases
export { default as SlashIcon } from '../icons/slash';
// Slack aliases
export { default as SlackIcon } from '../icons/slack';
// SmartphoneCharging aliases
export { default as SmartphoneChargingIcon } from '../icons/smartphone-charging';
// Slice aliases
export { default as SliceIcon } from '../icons/slice';
// SlidersHorizontal aliases
export { default as SlidersHorizontalIcon } from '../icons/sliders-horizontal';
// SmartphoneNfc aliases
export { default as SmartphoneNfcIcon } from '../icons/smartphone-nfc';
// Smartphone aliases
export { default as SmartphoneIcon } from '../icons/smartphone';
// Smile aliases
export { default as SmileIcon } from '../icons/smile';
// SmilePlus aliases
export { default as SmilePlusIcon } from '../icons/smile-plus';
// Snail aliases
export { default as SnailIcon } from '../icons/snail';
// Snowflake aliases
export { default as SnowflakeIcon } from '../icons/snowflake';
// SoapDispenserDroplet aliases
export { default as SoapDispenserDropletIcon } from '../icons/soap-dispenser-droplet';
// Sofa aliases
export { default as SofaIcon } from '../icons/sofa';
// Soup aliases
export { default as SoupIcon } from '../icons/soup';
// Spade aliases
export { default as SpadeIcon } from '../icons/spade';
// Space aliases
export { default as SpaceIcon } from '../icons/space';
// Speech aliases
export { default as SpeechIcon } from '../icons/speech';
// Speaker aliases
export { default as SpeakerIcon } from '../icons/speaker';
// Sparkle aliases
export { default as SparkleIcon } from '../icons/sparkle';
// SpellCheck2 aliases
export { default as SpellCheck2Icon } from '../icons/spell-check-2';
// SpellCheck aliases
export { default as SpellCheckIcon } from '../icons/spell-check';
// SplinePointer aliases
export { default as SplinePointerIcon } from '../icons/spline-pointer';
// Spline aliases
export { default as SplineIcon } from '../icons/spline';
// Spool aliases
export { default as SpoolIcon } from '../icons/spool';
// Spotlight aliases
export { default as SpotlightIcon } from '../icons/spotlight';
// Split aliases
export { default as SplitIcon } from '../icons/split';
// SprayCan aliases
export { default as SprayCanIcon } from '../icons/spray-can';
// Sprout aliases
export { default as SproutIcon } from '../icons/sprout';
// SquareDashedBottomCode aliases
export { default as SquareDashedBottomCodeIcon } from '../icons/square-dashed-bottom-code';
// SquareDashedBottom aliases
export { default as SquareDashedBottomIcon } from '../icons/square-dashed-bottom';
// SquareDashedTopSolid aliases
export { default as SquareDashedTopSolidIcon } from '../icons/square-dashed-top-solid';
// SquareRoundCorner aliases
export { default as SquareRoundCornerIcon } from '../icons/square-round-corner';
// SquarePause aliases
export { default as SquarePauseIcon } from '../icons/square-pause';
// SquareRadical aliases
export { default as SquareRadicalIcon } from '../icons/square-radical';
// SquareSquare aliases
export { default as SquareSquareIcon } from '../icons/square-square';
// SquareStack aliases
export { default as SquareStackIcon } from '../icons/square-stack';
// SquareStar aliases
export { default as SquareStarIcon } from '../icons/square-star';
// SquaresExclude aliases
export { default as SquaresExcludeIcon } from '../icons/squares-exclude';
// SquareStop aliases
export { default as SquareStopIcon } from '../icons/square-stop';
// Square aliases
export { default as SquareIcon } from '../icons/square';
// SquaresIntersect aliases
export { default as SquaresIntersectIcon } from '../icons/squares-intersect';
// SquaresSubtract aliases
export { default as SquaresSubtractIcon } from '../icons/squares-subtract';
// SquaresUnite aliases
export { default as SquaresUniteIcon } from '../icons/squares-unite';
// SquircleDashed aliases
export { default as SquircleDashedIcon } from '../icons/squircle-dashed';
// Squircle aliases
export { default as SquircleIcon } from '../icons/squircle';
// Squirrel aliases
export { default as SquirrelIcon } from '../icons/squirrel';
// Stamp aliases
export { default as StampIcon } from '../icons/stamp';
// StarHalf aliases
export { default as StarHalfIcon } from '../icons/star-half';
// Star aliases
export { default as StarIcon } from '../icons/star';
// StepBack aliases
export { default as StepBackIcon } from '../icons/step-back';
// StarOff aliases
export { default as StarOffIcon } from '../icons/star-off';
// StepForward aliases
export { default as StepForwardIcon } from '../icons/step-forward';
// Stethoscope aliases
export { default as StethoscopeIcon } from '../icons/stethoscope';
// Sticker aliases
export { default as StickerIcon } from '../icons/sticker';
// StickyNote aliases
export { default as StickyNoteIcon } from '../icons/sticky-note';
// Store aliases
export { default as StoreIcon } from '../icons/store';
// StretchHorizontal aliases
export { default as StretchHorizontalIcon } from '../icons/stretch-horizontal';
// StretchVertical aliases
export { default as StretchVerticalIcon } from '../icons/stretch-vertical';
// Subscript aliases
export { default as SubscriptIcon } from '../icons/subscript';
// Strikethrough aliases
export { default as StrikethroughIcon } from '../icons/strikethrough';
// SunDim aliases
export { default as SunDimIcon } from '../icons/sun-dim';
// SunMedium aliases
export { default as SunMediumIcon } from '../icons/sun-medium';
// SunMoon aliases
export { default as SunMoonIcon } from '../icons/sun-moon';
// SunSnow aliases
export { default as SunSnowIcon } from '../icons/sun-snow';
// Sun aliases
export { default as SunIcon } from '../icons/sun';
// Sunset aliases
export { default as SunsetIcon } from '../icons/sunset';
// Sunrise aliases
export { default as SunriseIcon } from '../icons/sunrise';
// SwatchBook aliases
export { default as SwatchBookIcon } from '../icons/swatch-book';
// Superscript aliases
export { default as SuperscriptIcon } from '../icons/superscript';
// SwissFranc aliases
export { default as SwissFrancIcon } from '../icons/swiss-franc';
// SwitchCamera aliases
export { default as SwitchCameraIcon } from '../icons/switch-camera';
// Sword aliases
export { default as SwordIcon } from '../icons/sword';
// Swords aliases
export { default as SwordsIcon } from '../icons/swords';
// Syringe aliases
export { default as SyringeIcon } from '../icons/syringe';
// Table2 aliases
export { default as Table2Icon } from '../icons/table-2';
// TableCellsMerge aliases
export { default as TableCellsMergeIcon } from '../icons/table-cells-merge';
// TableCellsSplit aliases
export { default as TableCellsSplitIcon } from '../icons/table-cells-split';
// TableColumnsSplit aliases
export { default as TableColumnsSplitIcon } from '../icons/table-columns-split';
// TableOfContents aliases
export { default as TableOfContentsIcon } from '../icons/table-of-contents';
// TableProperties aliases
export { default as TablePropertiesIcon } from '../icons/table-properties';
// TableRowsSplit aliases
export { default as TableRowsSplitIcon } from '../icons/table-rows-split';
// Table aliases
export { default as TableIcon } from '../icons/table';
// TabletSmartphone aliases
export { default as TabletSmartphoneIcon } from '../icons/tablet-smartphone';
// Tablet aliases
export { default as TabletIcon } from '../icons/tablet';
// Tablets aliases
export { default as TabletsIcon } from '../icons/tablets';
// Tag aliases
export { default as TagIcon } from '../icons/tag';
// Tags aliases
export { default as TagsIcon } from '../icons/tags';
// Tally1 aliases
export { default as Tally1Icon } from '../icons/tally-1';
// Tally2 aliases
export { default as Tally2Icon } from '../icons/tally-2';
// Tally3 aliases
export { default as Tally3Icon } from '../icons/tally-3';
// Tally4 aliases
export { default as Tally4Icon } from '../icons/tally-4';
// Tangent aliases
export { default as TangentIcon } from '../icons/tangent';
// Tally5 aliases
export { default as Tally5Icon } from '../icons/tally-5';
// Target aliases
export { default as TargetIcon } from '../icons/target';
// Telescope aliases
export { default as TelescopeIcon } from '../icons/telescope';
// Tent aliases
export { default as TentIcon } from '../icons/tent';
// TentTree aliases
export { default as TentTreeIcon } from '../icons/tent-tree';
// Terminal aliases
export { default as TerminalIcon } from '../icons/terminal';
// TextCursorInput aliases
export { default as TextCursorInputIcon } from '../icons/text-cursor-input';
// TestTube aliases
export { default as TestTubeIcon } from '../icons/test-tube';
// TestTubes aliases
export { default as TestTubesIcon } from '../icons/test-tubes';
// TextCursor aliases
export { default as TextCursorIcon } from '../icons/text-cursor';
// TextQuote aliases
export { default as TextQuoteIcon } from '../icons/text-quote';
// TextSearch aliases
export { default as TextSearchIcon } from '../icons/text-search';
// Theater aliases
export { default as TheaterIcon } from '../icons/theater';
// ThermometerSnowflake aliases
export { default as ThermometerSnowflakeIcon } from '../icons/thermometer-snowflake';
// ThermometerSun aliases
export { default as ThermometerSunIcon } from '../icons/thermometer-sun';
// ThumbsDown aliases
export { default as ThumbsDownIcon } from '../icons/thumbs-down';
// Thermometer aliases
export { default as ThermometerIcon } from '../icons/thermometer';
// TicketCheck aliases
export { default as TicketCheckIcon } from '../icons/ticket-check';
// TicketMinus aliases
export { default as TicketMinusIcon } from '../icons/ticket-minus';
// ThumbsUp aliases
export { default as ThumbsUpIcon } from '../icons/thumbs-up';
// TicketPercent aliases
export { default as TicketPercentIcon } from '../icons/ticket-percent';
// TicketPlus aliases
export { default as TicketPlusIcon } from '../icons/ticket-plus';
// TicketSlash aliases
export { default as TicketSlashIcon } from '../icons/ticket-slash';
// TicketX aliases
export { default as TicketXIcon } from '../icons/ticket-x';
// Ticket aliases
export { default as TicketIcon } from '../icons/ticket';
// TicketsPlane aliases
export { default as TicketsPlaneIcon } from '../icons/tickets-plane';
// TimerOff aliases
export { default as TimerOffIcon } from '../icons/timer-off';
// Tickets aliases
export { default as TicketsIcon } from '../icons/tickets';
// TimerReset aliases
export { default as TimerResetIcon } from '../icons/timer-reset';
// Timer aliases
export { default as TimerIcon } from '../icons/timer';
// ToggleLeft aliases
export { default as ToggleLeftIcon } from '../icons/toggle-left';
// Toilet aliases
export { default as ToiletIcon } from '../icons/toilet';
// ToolCase aliases
export { default as ToolCaseIcon } from '../icons/tool-case';
// ToggleRight aliases
export { default as ToggleRightIcon } from '../icons/toggle-right';
// Tornado aliases
export { default as TornadoIcon } from '../icons/tornado';
// Torus aliases
export { default as TorusIcon } from '../icons/torus';
// TouchpadOff aliases
export { default as TouchpadOffIcon } from '../icons/touchpad-off';
// Touchpad aliases
export { default as TouchpadIcon } from '../icons/touchpad';
// TowerControl aliases
export { default as TowerControlIcon } from '../icons/tower-control';
// ToyBrick aliases
export { default as ToyBrickIcon } from '../icons/toy-brick';
// Tractor aliases
export { default as TractorIcon } from '../icons/tractor';
// TrafficCone aliases
export { default as TrafficConeIcon } from '../icons/traffic-cone';
// TrainFront aliases
export { default as TrainFrontIcon } from '../icons/train-front';
// TrainFrontTunnel aliases
export { default as TrainFrontTunnelIcon } from '../icons/train-front-tunnel';
// TrainTrack aliases
export { default as TrainTrackIcon } from '../icons/train-track';
// Transgender aliases
export { default as TransgenderIcon } from '../icons/transgender';
// Trash2 aliases
export { default as Trash2Icon } from '../icons/trash-2';
// Trash aliases
export { default as TrashIcon } from '../icons/trash';
// TreeDeciduous aliases
export { default as TreeDeciduousIcon } from '../icons/tree-deciduous';
// TreePine aliases
export { default as TreePineIcon } from '../icons/tree-pine';
// Trello aliases
export { default as TrelloIcon } from '../icons/trello';
// Trees aliases
export { default as TreesIcon } from '../icons/trees';
// TrendingDown aliases
export { default as TrendingDownIcon } from '../icons/trending-down';
// TrendingUpDown aliases
export { default as TrendingUpDownIcon } from '../icons/trending-up-down';
// TrendingUp aliases
export { default as TrendingUpIcon } from '../icons/trending-up';
// TriangleDashed aliases
export { default as TriangleDashedIcon } from '../icons/triangle-dashed';
// Triangle aliases
export { default as TriangleIcon } from '../icons/triangle';
// TriangleRight aliases
export { default as TriangleRightIcon } from '../icons/triangle-right';
// Trophy aliases
export { default as TrophyIcon } from '../icons/trophy';
// Truck aliases
export { default as TruckIcon } from '../icons/truck';
// TruckElectric aliases
export { default as TruckElectricIcon } from '../icons/truck-electric';
// TurkishLira aliases
export { default as TurkishLiraIcon } from '../icons/turkish-lira';
// Turtle aliases
export { default as TurtleIcon } from '../icons/turtle';
// Turntable aliases
export { default as TurntableIcon } from '../icons/turntable';
// TvMinimalPlay aliases
export { default as TvMinimalPlayIcon } from '../icons/tv-minimal-play';
// Tv aliases
export { default as TvIcon } from '../icons/tv';
// Twitch aliases
export { default as TwitchIcon } from '../icons/twitch';
// Twitter aliases
export { default as TwitterIcon } from '../icons/twitter';
// TypeOutline aliases
export { default as TypeOutlineIcon } from '../icons/type-outline';
// Type aliases
export { default as TypeIcon } from '../icons/type';
// UmbrellaOff aliases
export { default as UmbrellaOffIcon } from '../icons/umbrella-off';
// Umbrella aliases
export { default as UmbrellaIcon } from '../icons/umbrella';
// Undo2 aliases
export { default as Undo2Icon } from '../icons/undo-2';
// Underline aliases
export { default as UnderlineIcon } from '../icons/underline';
// UndoDot aliases
export { default as UndoDotIcon } from '../icons/undo-dot';
// Undo aliases
export { default as UndoIcon } from '../icons/undo';
// Ungroup aliases
export { default as UngroupIcon } from '../icons/ungroup';
// UnfoldHorizontal aliases
export { default as UnfoldHorizontalIcon } from '../icons/unfold-horizontal';
// UnfoldVertical aliases
export { default as UnfoldVerticalIcon } from '../icons/unfold-vertical';
// Unlink aliases
export { default as UnlinkIcon } from '../icons/unlink';
// Unplug aliases
export { default as UnplugIcon } from '../icons/unplug';
// Unlink2 aliases
export { default as Unlink2Icon } from '../icons/unlink-2';
// Upload aliases
export { default as UploadIcon } from '../icons/upload';
// Usb aliases
export { default as UsbIcon } from '../icons/usb';
// UserCheck aliases
export { default as UserCheckIcon } from '../icons/user-check';
// UserCog aliases
export { default as UserCogIcon } from '../icons/user-cog';
// UserLock aliases
export { default as UserLockIcon } from '../icons/user-lock';
// UserMinus aliases
export { default as UserMinusIcon } from '../icons/user-minus';
// UserPlus aliases
export { default as UserPlusIcon } from '../icons/user-plus';
// UserPen aliases
export { default as UserPenIcon } from '../icons/user-pen';
// UserRoundPen aliases
export { default as UserRoundPenIcon } from '../icons/user-round-pen';
// UserRoundSearch aliases
export { default as UserRoundSearchIcon } from '../icons/user-round-search';
// UserSearch aliases
export { default as UserSearchIcon } from '../icons/user-search';
// UserStar aliases
export { default as UserStarIcon } from '../icons/user-star';
// UserX aliases
export { default as UserXIcon } from '../icons/user-x';
// User aliases
export { default as UserIcon } from '../icons/user';
// Users aliases
export { default as UsersIcon } from '../icons/users';
// UtilityPole aliases
export { default as UtilityPoleIcon } from '../icons/utility-pole';
// Variable aliases
export { default as VariableIcon } from '../icons/variable';
// Vault aliases
export { default as VaultIcon } from '../icons/vault';
// Vegan aliases
export { default as VeganIcon } from '../icons/vegan';
// VectorSquare aliases
export { default as VectorSquareIcon } from '../icons/vector-square';
// VenetianMask aliases
export { default as VenetianMaskIcon } from '../icons/venetian-mask';
// VenusAndMars aliases
export { default as VenusAndMarsIcon } from '../icons/venus-and-mars';
// Venus aliases
export { default as VenusIcon } from '../icons/venus';
// VibrateOff aliases
export { default as VibrateOffIcon } from '../icons/vibrate-off';
// Vibrate aliases
export { default as VibrateIcon } from '../icons/vibrate';
// VideoOff aliases
export { default as VideoOffIcon } from '../icons/video-off';
// Video aliases
export { default as VideoIcon } from '../icons/video';
// Videotape aliases
export { default as VideotapeIcon } from '../icons/videotape';
// View aliases
export { default as ViewIcon } from '../icons/view';
// Voicemail aliases
export { default as VoicemailIcon } from '../icons/voicemail';
// Volleyball aliases
export { default as VolleyballIcon } from '../icons/volleyball';
// Volume1 aliases
export { default as Volume1Icon } from '../icons/volume-1';
// Volume2 aliases
export { default as Volume2Icon } from '../icons/volume-2';
// VolumeOff aliases
export { default as VolumeOffIcon } from '../icons/volume-off';
// VolumeX aliases
export { default as VolumeXIcon } from '../icons/volume-x';
// Volume aliases
export { default as VolumeIcon } from '../icons/volume';
// WalletCards aliases
export { default as WalletCardsIcon } from '../icons/wallet-cards';
// Vote aliases
export { default as VoteIcon } from '../icons/vote';
// Wallpaper aliases
export { default as WallpaperIcon } from '../icons/wallpaper';
// Wallet aliases
export { default as WalletIcon } from '../icons/wallet';
// Wand aliases
export { default as WandIcon } from '../icons/wand';
// WashingMachine aliases
export { default as WashingMachineIcon } from '../icons/washing-machine';
// WavesLadder aliases
export { default as WavesLadderIcon } from '../icons/waves-ladder';
// Warehouse aliases
export { default as WarehouseIcon } from '../icons/warehouse';
// Watch aliases
export { default as WatchIcon } from '../icons/watch';
// Waves aliases
export { default as WavesIcon } from '../icons/waves';
// Waypoints aliases
export { default as WaypointsIcon } from '../icons/waypoints';
// WebhookOff aliases
export { default as WebhookOffIcon } from '../icons/webhook-off';
// Webcam aliases
export { default as WebcamIcon } from '../icons/webcam';
// Weight aliases
export { default as WeightIcon } from '../icons/weight';
// Webhook aliases
export { default as WebhookIcon } from '../icons/webhook';
// WheatOff aliases
export { default as WheatOffIcon } from '../icons/wheat-off';
// Wheat aliases
export { default as WheatIcon } from '../icons/wheat';
// WholeWord aliases
export { default as WholeWordIcon } from '../icons/whole-word';
// WifiCog aliases
export { default as WifiCogIcon } from '../icons/wifi-cog';
// WifiLow aliases
export { default as WifiLowIcon } from '../icons/wifi-low';
// WifiHigh aliases
export { default as WifiHighIcon } from '../icons/wifi-high';
// WifiOff aliases
export { default as WifiOffIcon } from '../icons/wifi-off';
// WifiPen aliases
export { default as WifiPenIcon } from '../icons/wifi-pen';
// WifiSync aliases
export { default as WifiSyncIcon } from '../icons/wifi-sync';
// WifiZero aliases
export { default as WifiZeroIcon } from '../icons/wifi-zero';
// Wifi aliases
export { default as WifiIcon } from '../icons/wifi';
// Wind aliases
export { default as WindIcon } from '../icons/wind';
// WindArrowDown aliases
export { default as WindArrowDownIcon } from '../icons/wind-arrow-down';
// WineOff aliases
export { default as WineOffIcon } from '../icons/wine-off';
// Wine aliases
export { default as WineIcon } from '../icons/wine';
// Workflow aliases
export { default as WorkflowIcon } from '../icons/workflow';
// Worm aliases
export { default as WormIcon } from '../icons/worm';
// Wrench aliases
export { default as WrenchIcon } from '../icons/wrench';
// X aliases
export { default as XIcon } from '../icons/x';
// Youtube aliases
export { default as YoutubeIcon } from '../icons/youtube';
// Zap aliases
export { default as ZapIcon } from '../icons/zap';
// ZapOff aliases
export { default as ZapOffIcon } from '../icons/zap-off';
// ZoomOut aliases
export { default as ZoomOutIcon } from '../icons/zoom-out';
// ZoomIn aliases
export { default as ZoomInIcon } from '../icons/zoom-in';
// ArrowDown01 aliases
export { default as ArrowDown01Icon } from '../icons/arrow-down-0-1';
// ArrowDown10 aliases
export { default as ArrowDown10Icon } from '../icons/arrow-down-1-0';
// ArrowUp01 aliases
export { default as ArrowUp01Icon } from '../icons/arrow-up-0-1';
// ArrowUp10 aliases
export { default as ArrowUp10Icon } from '../icons/arrow-up-1-0';
// AlarmClockCheck aliases
export { default as AlarmClockCheckIcon } from '../icons/alarm-clock-check';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link AlarmClockCheck} instead. This alias will be removed in v1.0 */
default as AlarmCheckIcon } from '../icons/alarm-clock-check';
// AlarmClockMinus aliases
export { default as AlarmClockMinusIcon } from '../icons/alarm-clock-minus';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link AlarmClockMinus} instead. This alias will be removed in v1.0 */
default as AlarmMinusIcon } from '../icons/alarm-clock-minus';
// AlarmClockPlus aliases
export { default as AlarmClockPlusIcon } from '../icons/alarm-clock-plus';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link AlarmClockPlus} instead. This alias will be removed in v1.0 */
default as AlarmPlusIcon } from '../icons/alarm-clock-plus';
// ArrowDownAZ aliases
export { default as ArrowDownAZIcon } from '../icons/arrow-down-a-z';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link ArrowDownAZ} instead. This alias will be removed in v1.0 */
default as ArrowDownAzIcon } from '../icons/arrow-down-a-z';
// ArrowDownWideNarrow aliases
export { default as ArrowDownWideNarrowIcon } from '../icons/arrow-down-wide-narrow';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link ArrowDownWideNarrow} instead. This alias will be removed in v1.0 */
default as SortDescIcon } from '../icons/arrow-down-wide-narrow';
// ArrowDownZA aliases
export { default as ArrowDownZAIcon } from '../icons/arrow-down-z-a';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link ArrowDownZA} instead. This alias will be removed in v1.0 */
default as ArrowDownZaIcon } from '../icons/arrow-down-z-a';
// ArrowUpNarrowWide aliases
export { default as ArrowUpNarrowWideIcon } from '../icons/arrow-up-narrow-wide';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link ArrowUpNarrowWide} instead. This alias will be removed in v1.0 */
default as SortAscIcon } from '../icons/arrow-up-narrow-wide';
// ArrowUpZA aliases
export { default as ArrowUpZAIcon } from '../icons/arrow-up-z-a';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link ArrowUpZA} instead. This alias will be removed in v1.0 */
default as ArrowUpZaIcon } from '../icons/arrow-up-z-a';
// ArrowUpAZ aliases
export { default as ArrowUpAZIcon } from '../icons/arrow-up-a-z';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link ArrowUpAZ} instead. This alias will be removed in v1.0 */
default as ArrowUpAzIcon } from '../icons/arrow-up-a-z';
// Axis3d aliases
export { default as Axis3dIcon } from '../icons/axis-3d';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link Axis3d} instead. This alias will be removed in v1.0 */
default as Axis3DIcon } from '../icons/axis-3d';
// BadgeCheck aliases
export { default as BadgeCheckIcon } from '../icons/badge-check';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link BadgeCheck} instead. This alias will be removed in v1.0 */
default as VerifiedIcon } from '../icons/badge-check';
// BadgeQuestionMark aliases
export { default as BadgeQuestionMarkIcon } from '../icons/badge-question-mark';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link BadgeQuestionMark} instead. This alias will be removed in v1.0 */
default as BadgeHelpIcon } from '../icons/badge-question-mark';
// BetweenHorizontalEnd aliases
export { default as BetweenHorizontalEndIcon } from '../icons/between-horizontal-end';
export { 
/** @deprecated Renamed because of typo, use {@link BetweenHorizontalEnd} instead. This alias will be removed in v1.0 */
default as BetweenHorizonalEndIcon } from '../icons/between-horizontal-end';
// BetweenHorizontalStart aliases
export { default as BetweenHorizontalStartIcon } from '../icons/between-horizontal-start';
export { 
/** @deprecated Renamed because of typo, use {@link BetweenHorizontalStart} instead. This alias will be removed in v1.0 */
default as BetweenHorizonalStartIcon } from '../icons/between-horizontal-start';
// Braces aliases
export { default as BracesIcon } from '../icons/braces';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link Braces} instead. This alias will be removed in v1.0 */
default as CurlyBracesIcon } from '../icons/braces';
// BookDashed aliases
export { default as BookDashedIcon } from '../icons/book-dashed';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link BookDashed} instead. This alias will be removed in v1.0.0 */
default as BookTemplateIcon } from '../icons/book-dashed';
// ChartArea aliases
export { default as ChartAreaIcon } from '../icons/chart-area';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link ChartArea} instead. This alias will be removed in v1.0 */
default as AreaChartIcon } from '../icons/chart-area';
// Captions aliases
export { default as CaptionsIcon } from '../icons/captions';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link Captions} instead. This alias will be removed in v1.0 */
default as SubtitlesIcon } from '../icons/captions';
// ChartBarBig aliases
export { default as ChartBarBigIcon } from '../icons/chart-bar-big';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link ChartBarBig} instead. This alias will be removed in v1.0 */
default as BarChartHorizontalBigIcon } from '../icons/chart-bar-big';
// ChartBar aliases
export { default as ChartBarIcon } from '../icons/chart-bar';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link ChartBar} instead. This alias will be removed in v1.0 */
default as BarChartHorizontalIcon } from '../icons/chart-bar';
// ChartCandlestick aliases
export { default as ChartCandlestickIcon } from '../icons/chart-candlestick';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link ChartCandlestick} instead. This alias will be removed in v1.0 */
default as CandlestickChartIcon } from '../icons/chart-candlestick';
// ChartColumnBig aliases
export { default as ChartColumnBigIcon } from '../icons/chart-column-big';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link ChartColumnBig} instead. This alias will be removed in v1.0 */
default as BarChartBigIcon } from '../icons/chart-column-big';
// ChartColumnIncreasing aliases
export { default as ChartColumnIncreasingIcon } from '../icons/chart-column-increasing';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link ChartColumnIncreasing} instead. This alias will be removed in v1.0 */
default as BarChart4Icon } from '../icons/chart-column-increasing';
// ChartColumn aliases
export { default as ChartColumnIcon } from '../icons/chart-column';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link ChartColumn} instead. This alias will be removed in v1.0 */
default as BarChart3Icon } from '../icons/chart-column';
// ChartLine aliases
export { default as ChartLineIcon } from '../icons/chart-line';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link ChartLine} instead. This alias will be removed in v1.0 */
default as LineChartIcon } from '../icons/chart-line';
// ChartNoAxesColumnIncreasing aliases
export { default as ChartNoAxesColumnIncreasingIcon } from '../icons/chart-no-axes-column-increasing';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link ChartNoAxesColumnIncreasing} instead. This alias will be removed in v1.0 */
default as BarChartIcon } from '../icons/chart-no-axes-column-increasing';
// ChartNoAxesColumn aliases
export { default as ChartNoAxesColumnIcon } from '../icons/chart-no-axes-column';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link ChartNoAxesColumn} instead. This alias will be removed in v1.0 */
default as BarChart2Icon } from '../icons/chart-no-axes-column';
// ChartPie aliases
export { default as ChartPieIcon } from '../icons/chart-pie';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link ChartPie} instead. This alias will be removed in v1.0 */
default as PieChartIcon } from '../icons/chart-pie';
// ChartNoAxesGantt aliases
export { default as ChartNoAxesGanttIcon } from '../icons/chart-no-axes-gantt';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link ChartNoAxesGantt} instead. This alias will be removed in v1.0 */
default as GanttChartIcon } from '../icons/chart-no-axes-gantt';
// ChartScatter aliases
export { default as ChartScatterIcon } from '../icons/chart-scatter';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link ChartScatter} instead. This alias will be removed in v1.0 */
default as ScatterChartIcon } from '../icons/chart-scatter';
// CircleAlert aliases
export { default as CircleAlertIcon } from '../icons/circle-alert';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CircleAlert} instead. This alias will be removed in v1.0 */
default as AlertCircleIcon } from '../icons/circle-alert';
// Chromium aliases
export { default as ChromiumIcon } from '../icons/chromium';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link Chromium} instead. This alias will be removed in v1.0 */
default as ChromeIcon } from '../icons/chromium';
// CircleArrowDown aliases
export { default as CircleArrowDownIcon } from '../icons/circle-arrow-down';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CircleArrowDown} instead. This alias will be removed in v1.0 */
default as ArrowDownCircleIcon } from '../icons/circle-arrow-down';
// CircleArrowLeft aliases
export { default as CircleArrowLeftIcon } from '../icons/circle-arrow-left';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CircleArrowLeft} instead. This alias will be removed in v1.0 */
default as ArrowLeftCircleIcon } from '../icons/circle-arrow-left';
// CircleArrowOutDownLeft aliases
export { default as CircleArrowOutDownLeftIcon } from '../icons/circle-arrow-out-down-left';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CircleArrowOutDownLeft} instead. This alias will be removed in v1.0 */
default as ArrowDownLeftFromCircleIcon } from '../icons/circle-arrow-out-down-left';
// CircleArrowOutDownRight aliases
export { default as CircleArrowOutDownRightIcon } from '../icons/circle-arrow-out-down-right';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CircleArrowOutDownRight} instead. This alias will be removed in v1.0 */
default as ArrowDownRightFromCircleIcon } from '../icons/circle-arrow-out-down-right';
// CircleArrowOutUpRight aliases
export { default as CircleArrowOutUpRightIcon } from '../icons/circle-arrow-out-up-right';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CircleArrowOutUpRight} instead. This alias will be removed in v1.0 */
default as ArrowUpRightFromCircleIcon } from '../icons/circle-arrow-out-up-right';
// CircleArrowOutUpLeft aliases
export { default as CircleArrowOutUpLeftIcon } from '../icons/circle-arrow-out-up-left';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CircleArrowOutUpLeft} instead. This alias will be removed in v1.0 */
default as ArrowUpLeftFromCircleIcon } from '../icons/circle-arrow-out-up-left';
// CircleArrowRight aliases
export { default as CircleArrowRightIcon } from '../icons/circle-arrow-right';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CircleArrowRight} instead. This alias will be removed in v1.0 */
default as ArrowRightCircleIcon } from '../icons/circle-arrow-right';
// CircleArrowUp aliases
export { default as CircleArrowUpIcon } from '../icons/circle-arrow-up';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CircleArrowUp} instead. This alias will be removed in v1.0 */
default as ArrowUpCircleIcon } from '../icons/circle-arrow-up';
// CircleCheckBig aliases
export { default as CircleCheckBigIcon } from '../icons/circle-check-big';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CircleCheckBig} instead. This alias will be removed in v1.0 */
default as CheckCircleIcon } from '../icons/circle-check-big';
// CircleChevronDown aliases
export { default as CircleChevronDownIcon } from '../icons/circle-chevron-down';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CircleChevronDown} instead. This alias will be removed in v1.0 */
default as ChevronDownCircleIcon } from '../icons/circle-chevron-down';
// CircleCheck aliases
export { default as CircleCheckIcon } from '../icons/circle-check';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CircleCheck} instead. This alias will be removed in v1.0 */
default as CheckCircle2Icon } from '../icons/circle-check';
// CircleChevronLeft aliases
export { default as CircleChevronLeftIcon } from '../icons/circle-chevron-left';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CircleChevronLeft} instead. This alias will be removed in v1.0 */
default as ChevronLeftCircleIcon } from '../icons/circle-chevron-left';
// CircleChevronRight aliases
export { default as CircleChevronRightIcon } from '../icons/circle-chevron-right';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CircleChevronRight} instead. This alias will be removed in v1.0 */
default as ChevronRightCircleIcon } from '../icons/circle-chevron-right';
// CircleDivide aliases
export { default as CircleDivideIcon } from '../icons/circle-divide';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CircleDivide} instead. This alias will be removed in v1.0 */
default as DivideCircleIcon } from '../icons/circle-divide';
// CircleGauge aliases
export { default as CircleGaugeIcon } from '../icons/circle-gauge';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CircleGauge} instead. This alias will be removed in v1.0 */
default as GaugeCircleIcon } from '../icons/circle-gauge';
// CircleChevronUp aliases
export { default as CircleChevronUpIcon } from '../icons/circle-chevron-up';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CircleChevronUp} instead. This alias will be removed in v1.0 */
default as ChevronUpCircleIcon } from '../icons/circle-chevron-up';
// CircleMinus aliases
export { default as CircleMinusIcon } from '../icons/circle-minus';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CircleMinus} instead. This alias will be removed in v1.0 */
default as MinusCircleIcon } from '../icons/circle-minus';
// CircleParking aliases
export { default as CircleParkingIcon } from '../icons/circle-parking';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CircleParking} instead. This alias will be removed in v1.0 */
default as ParkingCircleIcon } from '../icons/circle-parking';
// CirclePause aliases
export { default as CirclePauseIcon } from '../icons/circle-pause';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CirclePause} instead. This alias will be removed in v1.0 */
default as PauseCircleIcon } from '../icons/circle-pause';
// CircleParkingOff aliases
export { default as CircleParkingOffIcon } from '../icons/circle-parking-off';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CircleParkingOff} instead. This alias will be removed in v1.0 */
default as ParkingCircleOffIcon } from '../icons/circle-parking-off';
// CirclePlay aliases
export { default as CirclePlayIcon } from '../icons/circle-play';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CirclePlay} instead. This alias will be removed in v1.0 */
default as PlayCircleIcon } from '../icons/circle-play';
// CirclePercent aliases
export { default as CirclePercentIcon } from '../icons/circle-percent';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CirclePercent} instead. This alias will be removed in v1.0 */
default as PercentCircleIcon } from '../icons/circle-percent';
// CircleQuestionMark aliases
export { default as CircleQuestionMarkIcon } from '../icons/circle-question-mark';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CircleQuestionMark} instead. This alias will be removed in v1.0 */
default as HelpCircleIcon } from '../icons/circle-question-mark';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CircleQuestionMark} instead. This alias will be removed in v1.0 */
default as CircleHelpIcon } from '../icons/circle-question-mark';
// CirclePlus aliases
export { default as CirclePlusIcon } from '../icons/circle-plus';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CirclePlus} instead. This alias will be removed in v1.0 */
default as PlusCircleIcon } from '../icons/circle-plus';
// CircleSlash2 aliases
export { default as CircleSlash2Icon } from '../icons/circle-slash-2';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CircleSlash2} instead. This alias will be removed in v1.0 */
default as CircleSlashedIcon } from '../icons/circle-slash-2';
// CirclePower aliases
export { default as CirclePowerIcon } from '../icons/circle-power';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CirclePower} instead. This alias will be removed in v1.0 */
default as PowerCircleIcon } from '../icons/circle-power';
// CircleStop aliases
export { default as CircleStopIcon } from '../icons/circle-stop';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CircleStop} instead. This alias will be removed in v1.0 */
default as StopCircleIcon } from '../icons/circle-stop';
// CircleUserRound aliases
export { default as CircleUserRoundIcon } from '../icons/circle-user-round';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CircleUserRound} instead. This alias will be removed in v1.0 */
default as UserCircle2Icon } from '../icons/circle-user-round';
// CircleUser aliases
export { default as CircleUserIcon } from '../icons/circle-user';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CircleUser} instead. This alias will be removed in v1.0 */
default as UserCircleIcon } from '../icons/circle-user';
// CircleX aliases
export { default as CircleXIcon } from '../icons/circle-x';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CircleX} instead. This alias will be removed in v1.0 */
default as XCircleIcon } from '../icons/circle-x';
// ClipboardPenLine aliases
export { default as ClipboardPenLineIcon } from '../icons/clipboard-pen-line';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link ClipboardPenLine} instead. This alias will be removed in v1.0 */
default as ClipboardSignatureIcon } from '../icons/clipboard-pen-line';
// ClipboardPen aliases
export { default as ClipboardPenIcon } from '../icons/clipboard-pen';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link ClipboardPen} instead. This alias will be removed in v1.0 */
default as ClipboardEditIcon } from '../icons/clipboard-pen';
// CloudDownload aliases
export { default as CloudDownloadIcon } from '../icons/cloud-download';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CloudDownload} instead. This alias will be removed in v1.0 */
default as DownloadCloudIcon } from '../icons/cloud-download';
// CloudUpload aliases
export { default as CloudUploadIcon } from '../icons/cloud-upload';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CloudUpload} instead. This alias will be removed in v1.0 */
default as UploadCloudIcon } from '../icons/cloud-upload';
// CodeXml aliases
export { default as CodeXmlIcon } from '../icons/code-xml';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link CodeXml} instead. This alias will be removed in v1.0 */
default as Code2Icon } from '../icons/code-xml';
// Columns2 aliases
export { default as Columns2Icon } from '../icons/columns-2';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link Columns2} instead. This alias will be removed in v1.0 */
default as ColumnsIcon } from '../icons/columns-2';
// Columns3Cog aliases
export { default as Columns3CogIcon } from '../icons/columns-3-cog';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link Columns3Cog} instead. This alias will be removed in v1.0 */
default as ColumnsSettingsIcon } from '../icons/columns-3-cog';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link Columns3Cog} instead. This alias will be removed in v1.0 */
default as TableConfigIcon } from '../icons/columns-3-cog';
// ContactRound aliases
export { default as ContactRoundIcon } from '../icons/contact-round';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link ContactRound} instead. This alias will be removed in v1.0 */
default as Contact2Icon } from '../icons/contact-round';
// DiamondPercent aliases
export { default as DiamondPercentIcon } from '../icons/diamond-percent';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link DiamondPercent} instead. This alias will be removed in v1.0 */
default as PercentDiamondIcon } from '../icons/diamond-percent';
// Columns3 aliases
export { default as Columns3Icon } from '../icons/columns-3';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link Columns3} instead. This alias will be removed in v1.0 */
default as PanelsLeftRightIcon } from '../icons/columns-3';
// Earth aliases
export { default as EarthIcon } from '../icons/earth';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link Earth} instead. This alias will be removed in v1.0 */
default as Globe2Icon } from '../icons/earth';
// EllipsisVertical aliases
export { default as EllipsisVerticalIcon } from '../icons/ellipsis-vertical';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link EllipsisVertical} instead. This alias will be removed in v1.0 */
default as MoreVerticalIcon } from '../icons/ellipsis-vertical';
// Ellipsis aliases
export { default as EllipsisIcon } from '../icons/ellipsis';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link Ellipsis} instead. This alias will be removed in v1.0 */
default as MoreHorizontalIcon } from '../icons/ellipsis';
// FileAxis3d aliases
export { default as FileAxis3dIcon } from '../icons/file-axis-3d';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link FileAxis3d} instead. This alias will be removed in v1.0 */
default as FileAxis3DIcon } from '../icons/file-axis-3d';
// FileChartColumnIncreasing aliases
export { default as FileChartColumnIncreasingIcon } from '../icons/file-chart-column-increasing';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link FileChartColumnIncreasing} instead. This alias will be removed in v1.0 */
default as FileBarChartIcon } from '../icons/file-chart-column-increasing';
// FileChartLine aliases
export { default as FileChartLineIcon } from '../icons/file-chart-line';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link FileChartLine} instead. This alias will be removed in v1.0 */
default as FileLineChartIcon } from '../icons/file-chart-line';
// FileChartColumn aliases
export { default as FileChartColumnIcon } from '../icons/file-chart-column';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link FileChartColumn} instead. This alias will be removed in v1.0 */
default as FileBarChart2Icon } from '../icons/file-chart-column';
// FileChartPie aliases
export { default as FileChartPieIcon } from '../icons/file-chart-pie';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link FileChartPie} instead. This alias will be removed in v1.0 */
default as FilePieChartIcon } from '../icons/file-chart-pie';
// FilePenLine aliases
export { default as FilePenLineIcon } from '../icons/file-pen-line';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link FilePenLine} instead. This alias will be removed in v1.0 */
default as FileSignatureIcon } from '../icons/file-pen-line';
// FilePen aliases
export { default as FilePenIcon } from '../icons/file-pen';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link FilePen} instead. This alias will be removed in v1.0 */
default as FileEditIcon } from '../icons/file-pen';
// FileCog aliases
export { default as FileCogIcon } from '../icons/file-cog';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link FileCog} instead. This alias will be removed in v1.0 */
default as FileCog2Icon } from '../icons/file-cog';
// FileQuestionMark aliases
export { default as FileQuestionMarkIcon } from '../icons/file-question-mark';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link FileQuestionMark} instead. This alias will be removed in v1.0 */
default as FileQuestionIcon } from '../icons/file-question-mark';
// FileVideoCamera aliases
export { default as FileVideoCameraIcon } from '../icons/file-video-camera';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link FileVideoCamera} instead. This alias will be removed in v1.0 */
default as FileVideo2Icon } from '../icons/file-video-camera';
// FilePlay aliases
export { default as FilePlayIcon } from '../icons/file-play';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link FilePlay} instead. This alias will be removed in v1.0 */
default as FileVideoIcon } from '../icons/file-play';
// FolderCog aliases
export { default as FolderCogIcon } from '../icons/folder-cog';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link FolderCog} instead. This alias will be removed in v1.0 */
default as FolderCog2Icon } from '../icons/folder-cog';
// FolderPen aliases
export { default as FolderPenIcon } from '../icons/folder-pen';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link FolderPen} instead. This alias will be removed in v1.0 */
default as FolderEditIcon } from '../icons/folder-pen';
// FunnelX aliases
export { default as FunnelXIcon } from '../icons/funnel-x';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link FunnelX} instead. This alias will be removed in v1.0 */
default as FilterXIcon } from '../icons/funnel-x';
// Funnel aliases
export { default as FunnelIcon } from '../icons/funnel';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link Funnel} instead. This alias will be removed in v1.0 */
default as FilterIcon } from '../icons/funnel';
// GitCommitHorizontal aliases
export { default as GitCommitHorizontalIcon } from '../icons/git-commit-horizontal';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link GitCommitHorizontal} instead. This alias will be removed in v1.0 */
default as GitCommitIcon } from '../icons/git-commit-horizontal';
// Grid2x2Check aliases
export { default as Grid2x2CheckIcon } from '../icons/grid-2x2-check';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link Grid2x2Check} instead. This alias will be removed in v1.0 */
default as Grid2X2CheckIcon } from '../icons/grid-2x2-check';
// Grid2x2Plus aliases
export { default as Grid2x2PlusIcon } from '../icons/grid-2x2-plus';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link Grid2x2Plus} instead. This alias will be removed in v1.0 */
default as Grid2X2PlusIcon } from '../icons/grid-2x2-plus';
// Grid2x2X aliases
export { default as Grid2x2XIcon } from '../icons/grid-2x2-x';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link Grid2x2X} instead. This alias will be removed in v1.0 */
default as Grid2X2XIcon } from '../icons/grid-2x2-x';
// Grid2x2 aliases
export { default as Grid2x2Icon } from '../icons/grid-2x2';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link Grid2x2} instead. This alias will be removed in v1.0 */
default as Grid2X2Icon } from '../icons/grid-2x2';
// Grid3x3 aliases
export { default as Grid3x3Icon } from '../icons/grid-3x3';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link Grid3x3} instead. This alias will be removed in v1.0 */
default as GridIcon } from '../icons/grid-3x3';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link Grid3x3} instead. This alias will be removed in v1.0 */
default as Grid3X3Icon } from '../icons/grid-3x3';
// HandGrab aliases
export { default as HandGrabIcon } from '../icons/hand-grab';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link HandGrab} instead. This alias will be removed in v1.0 */
default as GrabIcon } from '../icons/hand-grab';
// House aliases
export { default as HouseIcon } from '../icons/house';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link House} instead. This alias will be removed in v1.0 */
default as HomeIcon } from '../icons/house';
// HandHelping aliases
export { default as HandHelpingIcon } from '../icons/hand-helping';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link HandHelping} instead. This alias will be removed in v1.0 */
default as HelpingHandIcon } from '../icons/hand-helping';
// IceCreamBowl aliases
export { default as IceCreamBowlIcon } from '../icons/ice-cream-bowl';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link IceCreamBowl} instead. This alias will be removed in v1.0 */
default as IceCream2Icon } from '../icons/ice-cream-bowl';
// IceCreamCone aliases
export { default as IceCreamConeIcon } from '../icons/ice-cream-cone';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link IceCreamCone} instead. This alias will be removed in v1.0 */
default as IceCreamIcon } from '../icons/ice-cream-cone';
// Layers aliases
export { default as LayersIcon } from '../icons/layers';
export { 
/** @deprecated The icon was combined with another icon that shares the same use case, use {@link Layers} instead. This alias will be removed in v1.0 */
default as Layers3Icon } from '../icons/layers';
// LaptopMinimal aliases
export { default as LaptopMinimalIcon } from '../icons/laptop-minimal';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link LaptopMinimal} instead. This alias will be removed in v1.0 */
default as Laptop2Icon } from '../icons/laptop-minimal';
// ListIndentDecrease aliases
export { default as ListIndentDecreaseIcon } from '../icons/list-indent-decrease';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link ListIndentDecrease} instead. This alias will be removed in v1.0 */
default as OutdentIcon } from '../icons/list-indent-decrease';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link ListIndentDecrease} instead. This alias will be removed in v1.0 */
default as IndentDecreaseIcon } from '../icons/list-indent-decrease';
// ListIndentIncrease aliases
export { default as ListIndentIncreaseIcon } from '../icons/list-indent-increase';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link ListIndentIncrease} instead. This alias will be removed in v1.0 */
default as IndentIcon } from '../icons/list-indent-increase';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link ListIndentIncrease} instead. This alias will be removed in v1.0 */
default as IndentIncreaseIcon } from '../icons/list-indent-increase';
// LoaderCircle aliases
export { default as LoaderCircleIcon } from '../icons/loader-circle';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link LoaderCircle} instead. This alias will be removed in v1.0 */
default as Loader2Icon } from '../icons/loader-circle';
// LockKeyholeOpen aliases
export { default as LockKeyholeOpenIcon } from '../icons/lock-keyhole-open';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link LockKeyholeOpen} instead. This alias will be removed in v1.0 */
default as UnlockKeyholeIcon } from '../icons/lock-keyhole-open';
// LockOpen aliases
export { default as LockOpenIcon } from '../icons/lock-open';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link LockOpen} instead. This alias will be removed in v1.0 */
default as UnlockIcon } from '../icons/lock-open';
// MailQuestionMark aliases
export { default as MailQuestionMarkIcon } from '../icons/mail-question-mark';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link MailQuestionMark} instead. This alias will be removed in v1.0 */
default as MailQuestionIcon } from '../icons/mail-question-mark';
// MessageCircleQuestionMark aliases
export { default as MessageCircleQuestionMarkIcon } from '../icons/message-circle-question-mark';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link MessageCircleQuestionMark} instead. This alias will be removed in v1.0 */
default as MessageCircleQuestionIcon } from '../icons/message-circle-question-mark';
// MicVocal aliases
export { default as MicVocalIcon } from '../icons/mic-vocal';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link MicVocal} instead. This alias will be removed in v1.0 */
default as Mic2Icon } from '../icons/mic-vocal';
// MapPinPen aliases
export { default as MapPinPenIcon } from '../icons/map-pin-pen';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link MapPinPen} instead. This alias will be removed in v1.0 */
default as LocationEditIcon } from '../icons/map-pin-pen';
// Move3d aliases
export { default as Move3dIcon } from '../icons/move-3d';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link Move3d} instead. This alias will be removed in v1.0 */
default as Move3DIcon } from '../icons/move-3d';
// OctagonAlert aliases
export { default as OctagonAlertIcon } from '../icons/octagon-alert';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link OctagonAlert} instead. This alias will be removed in v1.0 */
default as AlertOctagonIcon } from '../icons/octagon-alert';
// OctagonX aliases
export { default as OctagonXIcon } from '../icons/octagon-x';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link OctagonX} instead. This alias will be removed in v1.0 */
default as XOctagonIcon } from '../icons/octagon-x';
// OctagonPause aliases
export { default as OctagonPauseIcon } from '../icons/octagon-pause';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link OctagonPause} instead. This alias will be removed in v1.0 */
default as PauseOctagonIcon } from '../icons/octagon-pause';
// PaintbrushVertical aliases
export { default as PaintbrushVerticalIcon } from '../icons/paintbrush-vertical';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link PaintbrushVertical} instead. This alias will be removed in v1.0 */
default as Paintbrush2Icon } from '../icons/paintbrush-vertical';
// PanelLeftClose aliases
export { default as PanelLeftCloseIcon } from '../icons/panel-left-close';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link PanelLeftClose} instead. This alias will be removed in v1.0 */
default as SidebarCloseIcon } from '../icons/panel-left-close';
// PanelBottomDashed aliases
export { default as PanelBottomDashedIcon } from '../icons/panel-bottom-dashed';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link PanelBottomDashed} instead. This alias will be removed in v1.0 */
default as PanelBottomInactiveIcon } from '../icons/panel-bottom-dashed';
// PanelLeftOpen aliases
export { default as PanelLeftOpenIcon } from '../icons/panel-left-open';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link PanelLeftOpen} instead. This alias will be removed in v1.0 */
default as SidebarOpenIcon } from '../icons/panel-left-open';
// PanelLeftDashed aliases
export { default as PanelLeftDashedIcon } from '../icons/panel-left-dashed';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link PanelLeftDashed} instead. This alias will be removed in v1.0 */
default as PanelLeftInactiveIcon } from '../icons/panel-left-dashed';
// PanelLeft aliases
export { default as PanelLeftIcon } from '../icons/panel-left';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link PanelLeft} instead. This alias will be removed in v1.0 */
default as SidebarIcon } from '../icons/panel-left';
// PanelRightDashed aliases
export { default as PanelRightDashedIcon } from '../icons/panel-right-dashed';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link PanelRightDashed} instead. This alias will be removed in v1.0 */
default as PanelRightInactiveIcon } from '../icons/panel-right-dashed';
// PanelTopDashed aliases
export { default as PanelTopDashedIcon } from '../icons/panel-top-dashed';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link PanelTopDashed} instead. This alias will be removed in v1.0 */
default as PanelTopInactiveIcon } from '../icons/panel-top-dashed';
// PanelsTopLeft aliases
export { default as PanelsTopLeftIcon } from '../icons/panels-top-left';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link PanelsTopLeft} instead. This alias will be removed in v1.0 */
default as LayoutIcon } from '../icons/panels-top-left';
// PenLine aliases
export { default as PenLineIcon } from '../icons/pen-line';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link PenLine} instead. This alias will be removed in v1.0 */
default as Edit3Icon } from '../icons/pen-line';
// Pen aliases
export { default as PenIcon } from '../icons/pen';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link Pen} instead. This alias will be removed in v1.0 */
default as Edit2Icon } from '../icons/pen';
// PlugZap aliases
export { default as PlugZapIcon } from '../icons/plug-zap';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link PlugZap} instead. This alias will be removed in v1.0 */
default as PlugZap2Icon } from '../icons/plug-zap';
// RectangleEllipsis aliases
export { default as RectangleEllipsisIcon } from '../icons/rectangle-ellipsis';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link RectangleEllipsis} instead. This alias will be removed in v1.0 */
default as FormInputIcon } from '../icons/rectangle-ellipsis';
// Rotate3d aliases
export { default as Rotate3dIcon } from '../icons/rotate-3d';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link Rotate3d} instead. This alias will be removed in v1.0 */
default as Rotate3DIcon } from '../icons/rotate-3d';
// Rows2 aliases
export { default as Rows2Icon } from '../icons/rows-2';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link Rows2} instead. This alias will be removed in v1.0 */
default as RowsIcon } from '../icons/rows-2';
// Rows3 aliases
export { default as Rows3Icon } from '../icons/rows-3';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link Rows3} instead. This alias will be removed in v1.0 */
default as PanelsTopBottomIcon } from '../icons/rows-3';
// SendHorizontal aliases
export { default as SendHorizontalIcon } from '../icons/send-horizontal';
export { 
/** @deprecated Renamed because of typo, use {@link SendHorizontal} instead. This alias will be removed in v1.0 */
default as SendHorizonalIcon } from '../icons/send-horizontal';
// ShieldQuestionMark aliases
export { default as ShieldQuestionMarkIcon } from '../icons/shield-question-mark';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link ShieldQuestionMark} instead. This alias will be removed in v1.0 */
default as ShieldQuestionIcon } from '../icons/shield-question-mark';
// Scale3d aliases
export { default as Scale3dIcon } from '../icons/scale-3d';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link Scale3d} instead. This alias will be removed in v1.0 */
default as Scale3DIcon } from '../icons/scale-3d';
// SlidersVertical aliases
export { default as SlidersVerticalIcon } from '../icons/sliders-vertical';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SlidersVertical} instead. This alias will be removed in v1.0 */
default as SlidersIcon } from '../icons/sliders-vertical';
// ShieldX aliases
export { default as ShieldXIcon } from '../icons/shield-x';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link ShieldX} instead. This alias will be removed in v1.0 */
default as ShieldCloseIcon } from '../icons/shield-x';
// Sparkles aliases
export { default as SparklesIcon } from '../icons/sparkles';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link Sparkles} instead. This alias will be removed in v1.0 */
default as StarsIcon } from '../icons/sparkles';
// SquareActivity aliases
export { default as SquareActivityIcon } from '../icons/square-activity';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareActivity} instead. This alias will be removed in v1.0 */
default as ActivitySquareIcon } from '../icons/square-activity';
// SquareArrowDownLeft aliases
export { default as SquareArrowDownLeftIcon } from '../icons/square-arrow-down-left';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareArrowDownLeft} instead. This alias will be removed in v1.0 */
default as ArrowDownLeftSquareIcon } from '../icons/square-arrow-down-left';
// SquareArrowDownRight aliases
export { default as SquareArrowDownRightIcon } from '../icons/square-arrow-down-right';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareArrowDownRight} instead. This alias will be removed in v1.0 */
default as ArrowDownRightSquareIcon } from '../icons/square-arrow-down-right';
// SquareArrowDown aliases
export { default as SquareArrowDownIcon } from '../icons/square-arrow-down';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareArrowDown} instead. This alias will be removed in v1.0 */
default as ArrowDownSquareIcon } from '../icons/square-arrow-down';
// SquareArrowOutDownRight aliases
export { default as SquareArrowOutDownRightIcon } from '../icons/square-arrow-out-down-right';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareArrowOutDownRight} instead. This alias will be removed in v1.0 */
default as ArrowDownRightFromSquareIcon } from '../icons/square-arrow-out-down-right';
// SquareArrowLeft aliases
export { default as SquareArrowLeftIcon } from '../icons/square-arrow-left';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareArrowLeft} instead. This alias will be removed in v1.0 */
default as ArrowLeftSquareIcon } from '../icons/square-arrow-left';
// SquareArrowOutDownLeft aliases
export { default as SquareArrowOutDownLeftIcon } from '../icons/square-arrow-out-down-left';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareArrowOutDownLeft} instead. This alias will be removed in v1.0 */
default as ArrowDownLeftFromSquareIcon } from '../icons/square-arrow-out-down-left';
// SquareArrowOutUpLeft aliases
export { default as SquareArrowOutUpLeftIcon } from '../icons/square-arrow-out-up-left';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareArrowOutUpLeft} instead. This alias will be removed in v1.0 */
default as ArrowUpLeftFromSquareIcon } from '../icons/square-arrow-out-up-left';
// SquareArrowRight aliases
export { default as SquareArrowRightIcon } from '../icons/square-arrow-right';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareArrowRight} instead. This alias will be removed in v1.0 */
default as ArrowRightSquareIcon } from '../icons/square-arrow-right';
// SquareArrowOutUpRight aliases
export { default as SquareArrowOutUpRightIcon } from '../icons/square-arrow-out-up-right';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareArrowOutUpRight} instead. This alias will be removed in v1.0 */
default as ArrowUpRightFromSquareIcon } from '../icons/square-arrow-out-up-right';
// SquareArrowUpRight aliases
export { default as SquareArrowUpRightIcon } from '../icons/square-arrow-up-right';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareArrowUpRight} instead. This alias will be removed in v1.0 */
default as ArrowUpRightSquareIcon } from '../icons/square-arrow-up-right';
// SquareArrowUpLeft aliases
export { default as SquareArrowUpLeftIcon } from '../icons/square-arrow-up-left';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareArrowUpLeft} instead. This alias will be removed in v1.0 */
default as ArrowUpLeftSquareIcon } from '../icons/square-arrow-up-left';
// SquareArrowUp aliases
export { default as SquareArrowUpIcon } from '../icons/square-arrow-up';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareArrowUp} instead. This alias will be removed in v1.0 */
default as ArrowUpSquareIcon } from '../icons/square-arrow-up';
// SquareAsterisk aliases
export { default as SquareAsteriskIcon } from '../icons/square-asterisk';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareAsterisk} instead. This alias will be removed in v1.0 */
default as AsteriskSquareIcon } from '../icons/square-asterisk';
// SquareChartGantt aliases
export { default as SquareChartGanttIcon } from '../icons/square-chart-gantt';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareChartGantt} instead. This alias will be removed in v1.0 */
default as GanttChartSquareIcon } from '../icons/square-chart-gantt';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareChartGantt} instead. This alias will be removed in v1.0 */
default as SquareGanttChartIcon } from '../icons/square-chart-gantt';
// SquareCheckBig aliases
export { default as SquareCheckBigIcon } from '../icons/square-check-big';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareCheckBig} instead. This alias will be removed in v1.0 */
default as CheckSquareIcon } from '../icons/square-check-big';
// SquareBottomDashedScissors aliases
export { default as SquareBottomDashedScissorsIcon } from '../icons/square-bottom-dashed-scissors';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareBottomDashedScissors} instead. This alias will be removed in v1.0 */
default as ScissorsSquareDashedBottomIcon } from '../icons/square-bottom-dashed-scissors';
// SquareCheck aliases
export { default as SquareCheckIcon } from '../icons/square-check';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareCheck} instead. This alias will be removed in v1.0 */
default as CheckSquare2Icon } from '../icons/square-check';
// SquareChevronLeft aliases
export { default as SquareChevronLeftIcon } from '../icons/square-chevron-left';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareChevronLeft} instead. This alias will be removed in v1.0 */
default as ChevronLeftSquareIcon } from '../icons/square-chevron-left';
// SquareChevronDown aliases
export { default as SquareChevronDownIcon } from '../icons/square-chevron-down';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareChevronDown} instead. This alias will be removed in v1.0 */
default as ChevronDownSquareIcon } from '../icons/square-chevron-down';
// SquareChevronRight aliases
export { default as SquareChevronRightIcon } from '../icons/square-chevron-right';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareChevronRight} instead. This alias will be removed in v1.0 */
default as ChevronRightSquareIcon } from '../icons/square-chevron-right';
// SquareChevronUp aliases
export { default as SquareChevronUpIcon } from '../icons/square-chevron-up';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareChevronUp} instead. This alias will be removed in v1.0 */
default as ChevronUpSquareIcon } from '../icons/square-chevron-up';
// SquareCode aliases
export { default as SquareCodeIcon } from '../icons/square-code';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareCode} instead. This alias will be removed in v1.0 */
default as CodeSquareIcon } from '../icons/square-code';
// SquareDashedKanban aliases
export { default as SquareDashedKanbanIcon } from '../icons/square-dashed-kanban';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareDashedKanban} instead. This alias will be removed in v1.0 */
default as KanbanSquareDashedIcon } from '../icons/square-dashed-kanban';
// SquareDashedMousePointer aliases
export { default as SquareDashedMousePointerIcon } from '../icons/square-dashed-mouse-pointer';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareDashedMousePointer} instead. This alias will be removed in v1.0 */
default as MousePointerSquareDashedIcon } from '../icons/square-dashed-mouse-pointer';
// SquareDashed aliases
export { default as SquareDashedIcon } from '../icons/square-dashed';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareDashed} instead. This alias will be removed in v1.0 */
default as BoxSelectIcon } from '../icons/square-dashed';
// SquareDot aliases
export { default as SquareDotIcon } from '../icons/square-dot';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareDot} instead. This alias will be removed in v1.0 */
default as DotSquareIcon } from '../icons/square-dot';
// SquareEqual aliases
export { default as SquareEqualIcon } from '../icons/square-equal';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareEqual} instead. This alias will be removed in v1.0 */
default as EqualSquareIcon } from '../icons/square-equal';
// SquareDivide aliases
export { default as SquareDivideIcon } from '../icons/square-divide';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareDivide} instead. This alias will be removed in v1.0 */
default as DivideSquareIcon } from '../icons/square-divide';
// SquareFunction aliases
export { default as SquareFunctionIcon } from '../icons/square-function';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareFunction} instead. This alias will be removed in v1.0 */
default as FunctionSquareIcon } from '../icons/square-function';
// SquareKanban aliases
export { default as SquareKanbanIcon } from '../icons/square-kanban';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareKanban} instead. This alias will be removed in v1.0 */
default as KanbanSquareIcon } from '../icons/square-kanban';
// SquareLibrary aliases
export { default as SquareLibraryIcon } from '../icons/square-library';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareLibrary} instead. This alias will be removed in v1.0 */
default as LibrarySquareIcon } from '../icons/square-library';
// SquareM aliases
export { default as SquareMIcon } from '../icons/square-m';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareM} instead. This alias will be removed in v1.0 */
default as MSquareIcon } from '../icons/square-m';
// SquareMousePointer aliases
export { default as SquareMousePointerIcon } from '../icons/square-mouse-pointer';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareMousePointer} instead. This alias will be removed in v1.0 */
default as InspectIcon } from '../icons/square-mouse-pointer';
// SquareMenu aliases
export { default as SquareMenuIcon } from '../icons/square-menu';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareMenu} instead. This alias will be removed in v1.0 */
default as MenuSquareIcon } from '../icons/square-menu';
// SquareParkingOff aliases
export { default as SquareParkingOffIcon } from '../icons/square-parking-off';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareParkingOff} instead. This alias will be removed in v1.0 */
default as ParkingSquareOffIcon } from '../icons/square-parking-off';
// SquareMinus aliases
export { default as SquareMinusIcon } from '../icons/square-minus';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareMinus} instead. This alias will be removed in v1.0 */
default as MinusSquareIcon } from '../icons/square-minus';
// SquarePen aliases
export { default as SquarePenIcon } from '../icons/square-pen';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquarePen} instead. This alias will be removed in v1.0 */
default as PenBoxIcon } from '../icons/square-pen';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquarePen} instead. This alias will be removed in v1.0 */
default as EditIcon } from '../icons/square-pen';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquarePen} instead. This alias will be removed in v1.0 */
default as PenSquareIcon } from '../icons/square-pen';
// SquareParking aliases
export { default as SquareParkingIcon } from '../icons/square-parking';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareParking} instead. This alias will be removed in v1.0 */
default as ParkingSquareIcon } from '../icons/square-parking';
// SquarePercent aliases
export { default as SquarePercentIcon } from '../icons/square-percent';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquarePercent} instead. This alias will be removed in v1.0 */
default as PercentSquareIcon } from '../icons/square-percent';
// SquarePi aliases
export { default as SquarePiIcon } from '../icons/square-pi';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquarePi} instead. This alias will be removed in v1.0 */
default as PiSquareIcon } from '../icons/square-pi';
// SquarePilcrow aliases
export { default as SquarePilcrowIcon } from '../icons/square-pilcrow';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquarePilcrow} instead. This alias will be removed in v1.0 */
default as PilcrowSquareIcon } from '../icons/square-pilcrow';
// SquarePlay aliases
export { default as SquarePlayIcon } from '../icons/square-play';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquarePlay} instead. This alias will be removed in v1.0 */
default as PlaySquareIcon } from '../icons/square-play';
// SquarePlus aliases
export { default as SquarePlusIcon } from '../icons/square-plus';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquarePlus} instead. This alias will be removed in v1.0 */
default as PlusSquareIcon } from '../icons/square-plus';
// SquarePower aliases
export { default as SquarePowerIcon } from '../icons/square-power';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquarePower} instead. This alias will be removed in v1.0 */
default as PowerSquareIcon } from '../icons/square-power';
// SquareScissors aliases
export { default as SquareScissorsIcon } from '../icons/square-scissors';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareScissors} instead. This alias will be removed in v1.0 */
default as ScissorsSquareIcon } from '../icons/square-scissors';
// SquareSigma aliases
export { default as SquareSigmaIcon } from '../icons/square-sigma';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareSigma} instead. This alias will be removed in v1.0 */
default as SigmaSquareIcon } from '../icons/square-sigma';
// SquareSplitHorizontal aliases
export { default as SquareSplitHorizontalIcon } from '../icons/square-split-horizontal';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareSplitHorizontal} instead. This alias will be removed in v1.0 */
default as SplitSquareHorizontalIcon } from '../icons/square-split-horizontal';
// SquareSplitVertical aliases
export { default as SquareSplitVerticalIcon } from '../icons/square-split-vertical';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareSplitVertical} instead. This alias will be removed in v1.0 */
default as SplitSquareVerticalIcon } from '../icons/square-split-vertical';
// SquareSlash aliases
export { default as SquareSlashIcon } from '../icons/square-slash';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareSlash} instead. This alias will be removed in v1.0 */
default as SlashSquareIcon } from '../icons/square-slash';
// SquareTerminal aliases
export { default as SquareTerminalIcon } from '../icons/square-terminal';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareTerminal} instead. This alias will be removed in v1.0 */
default as TerminalSquareIcon } from '../icons/square-terminal';
// SquareUserRound aliases
export { default as SquareUserRoundIcon } from '../icons/square-user-round';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareUserRound} instead. This alias will be removed in v1.0 */
default as UserSquare2Icon } from '../icons/square-user-round';
// SquareUser aliases
export { default as SquareUserIcon } from '../icons/square-user';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareUser} instead. This alias will be removed in v1.0 */
default as UserSquareIcon } from '../icons/square-user';
// SquareX aliases
export { default as SquareXIcon } from '../icons/square-x';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link SquareX} instead. This alias will be removed in v1.0 */
default as XSquareIcon } from '../icons/square-x';
// TestTubeDiagonal aliases
export { default as TestTubeDiagonalIcon } from '../icons/test-tube-diagonal';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link TestTubeDiagonal} instead. This alias will be removed in v1.0 */
default as TestTube2Icon } from '../icons/test-tube-diagonal';
// TextAlignCenter aliases
export { default as TextAlignCenterIcon } from '../icons/text-align-center';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link TextAlignCenter} instead. This alias will be removed in v1.0 */
default as AlignCenterIcon } from '../icons/text-align-center';
// TextAlignEnd aliases
export { default as TextAlignEndIcon } from '../icons/text-align-end';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link TextAlignEnd} instead. This alias will be removed in v1.0 */
default as AlignRightIcon } from '../icons/text-align-end';
// TextAlignStart aliases
export { default as TextAlignStartIcon } from '../icons/text-align-start';
export { 
/** @deprecated The icon was combined with another icon that shares the same use case, use {@link TextAlignStart} instead. This alias will be removed in v1.0 */
default as TextIcon } from '../icons/text-align-start';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link TextAlignStart} instead. This alias will be removed in v1.0 */
default as AlignLeftIcon } from '../icons/text-align-start';
// TextAlignJustify aliases
export { default as TextAlignJustifyIcon } from '../icons/text-align-justify';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link TextAlignJustify} instead. This alias will be removed in v1.0 */
default as AlignJustifyIcon } from '../icons/text-align-justify';
// TextSelect aliases
export { default as TextSelectIcon } from '../icons/text-select';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link TextSelect} instead. This alias will be removed in v1.0 */
default as TextSelectionIcon } from '../icons/text-select';
// TextWrap aliases
export { default as TextWrapIcon } from '../icons/text-wrap';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link TextWrap} instead. This alias will be removed in v1.0 */
default as WrapTextIcon } from '../icons/text-wrap';
// TextInitial aliases
export { default as TextInitialIcon } from '../icons/text-initial';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link TextInitial} instead. This alias will be removed in v1.0 */
default as LetterTextIcon } from '../icons/text-initial';
// TramFront aliases
export { default as TramFrontIcon } from '../icons/tram-front';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link TramFront} instead. This alias will be removed in v1.0 */
default as TrainIcon } from '../icons/tram-front';
// TreePalm aliases
export { default as TreePalmIcon } from '../icons/tree-palm';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link TreePalm} instead. This alias will be removed in v1.0 */
default as PalmtreeIcon } from '../icons/tree-palm';
// TriangleAlert aliases
export { default as TriangleAlertIcon } from '../icons/triangle-alert';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link TriangleAlert} instead. This alias will be removed in v1.0 */
default as AlertTriangleIcon } from '../icons/triangle-alert';
// TvMinimal aliases
export { default as TvMinimalIcon } from '../icons/tv-minimal';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link TvMinimal} instead. This alias will be removed in v1.0 */
default as Tv2Icon } from '../icons/tv-minimal';
// UserRoundCog aliases
export { default as UserRoundCogIcon } from '../icons/user-round-cog';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link UserRoundCog} instead. This alias will be removed in v1.0 */
default as UserCog2Icon } from '../icons/user-round-cog';
// University aliases
export { default as UniversityIcon } from '../icons/university';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link University} instead. This alias will be removed in v1.0 */
default as School2Icon } from '../icons/university';
// UserRoundCheck aliases
export { default as UserRoundCheckIcon } from '../icons/user-round-check';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link UserRoundCheck} instead. This alias will be removed in v1.0 */
default as UserCheck2Icon } from '../icons/user-round-check';
// UserRoundPlus aliases
export { default as UserRoundPlusIcon } from '../icons/user-round-plus';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link UserRoundPlus} instead. This alias will be removed in v1.0 */
default as UserPlus2Icon } from '../icons/user-round-plus';
// UserRoundMinus aliases
export { default as UserRoundMinusIcon } from '../icons/user-round-minus';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link UserRoundMinus} instead. This alias will be removed in v1.0 */
default as UserMinus2Icon } from '../icons/user-round-minus';
// UserRound aliases
export { default as UserRoundIcon } from '../icons/user-round';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link UserRound} instead. This alias will be removed in v1.0 */
default as User2Icon } from '../icons/user-round';
// UserRoundX aliases
export { default as UserRoundXIcon } from '../icons/user-round-x';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link UserRoundX} instead. This alias will be removed in v1.0 */
default as UserX2Icon } from '../icons/user-round-x';
// UsersRound aliases
export { default as UsersRoundIcon } from '../icons/users-round';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link UsersRound} instead. This alias will be removed in v1.0 */
default as Users2Icon } from '../icons/users-round';
// UtensilsCrossed aliases
export { default as UtensilsCrossedIcon } from '../icons/utensils-crossed';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link UtensilsCrossed} instead. This alias will be removed in v1.0 */
default as ForkKnifeCrossedIcon } from '../icons/utensils-crossed';
// Utensils aliases
export { default as UtensilsIcon } from '../icons/utensils';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link Utensils} instead. This alias will be removed in v1.0 */
default as ForkKnifeIcon } from '../icons/utensils';
// WalletMinimal aliases
export { default as WalletMinimalIcon } from '../icons/wallet-minimal';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link WalletMinimal} instead. This alias will be removed in v1.0 */
default as Wallet2Icon } from '../icons/wallet-minimal';
// WandSparkles aliases
export { default as WandSparklesIcon } from '../icons/wand-sparkles';
export { 
/** @deprecated The name of this icon was changed because it didn't meet our guidelines anymore, use {@link WandSparkles} instead. This alias will be removed in v1.0 */
default as Wand2Icon } from '../icons/wand-sparkles';
